<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.335">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="M.Lagisz">

<title>Learning bibliometrix</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<script src="learn_bibliometrix_files/libs/clipboard/clipboard.min.js"></script>
<script src="learn_bibliometrix_files/libs/quarto-html/quarto.js"></script>
<script src="learn_bibliometrix_files/libs/quarto-html/popper.min.js"></script>
<script src="learn_bibliometrix_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="learn_bibliometrix_files/libs/quarto-html/anchor.min.js"></script>
<link href="learn_bibliometrix_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="learn_bibliometrix_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="learn_bibliometrix_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="learn_bibliometrix_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="learn_bibliometrix_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<div class="quarto-title-block"><div><h1 class="title">Learning bibliometrix</h1><button type="button" class="btn code-tools-button dropdown-toggle" id="quarto-code-tools-menu" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi"></i> Code</button><ul class="dropdown-menu dropdown-menu-end" aria-labelelledby="quarto-code-tools-menu"><li><a id="quarto-show-all-code" class="dropdown-item" href="javascript:void(0)" role="button">Show All Code</a></li><li><a id="quarto-hide-all-code" class="dropdown-item" href="javascript:void(0)" role="button">Hide All Code</a></li><li><hr class="dropdown-divider"></li><li><a id="quarto-view-source" class="dropdown-item" href="javascript:void(0)" role="button">View Source</a></li></ul></div></div>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>M.Lagisz </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">5/31/23</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<p><strong>Before the lesson:</strong><br>
Please make sure you got the latest RStudio and latest R version installed.</p>
<p><strong>Lesson objectives:</strong><br>
* Learn to perform a search in academic literature database<br>
* Download search results and import them into R<br>
* Summarise bibliometric data<br>
* Make a few types of simple bibliometric networks<br>
* Plot bibliometric networks</p>
<p><strong>Lesson outline:</strong><br>
* About this lesson<br>
* Getting bibliometric data<br>
* Summarising bibliometric data<br>
* Creating bibliometric networks<br>
* More resources</p>
<hr>
<section id="about-this-lesson" class="level2">
<h2 class="anchored" data-anchor-id="about-this-lesson">About this lesson</h2>
<p>This lesson is prepared for these who are already familiar with R coding language, R markdown and RStudio. By the end of this tutorial you should be able to create a simple html document containing markdown-formatted text, images and R code - all in RStudio.</p>
<hr>
</section>
<section id="getting-bibliometric-data" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="getting-bibliometric-data">Getting bibliometric data</h2>
<p>You can do analyses of literature on any topic.</p>
<p>In this lesson we will have a look at the academic literature related to the concept of Terminal Investment.</p>
<p>Terminal Investment hypothesis predicts increased investment of resources into reproduction as the chances of survival decrease. This can be observed as increased reproductive effort in older animals or in animals challenged with factors signalling threat to their survival (e.g., predation, pathogenes, parasites).</p>
<p>Terminal investment in animals is usually studied in three main ways:<br>
1. via observational studies of correlations of age and reproductive effort,<br>
2. in experimental studies where animals are subject to immune challenges and their subsequent reproductive effort is compared to unchallenged animals of the same age,<br>
3. in experimental studies where reproductive response to immune challenge is compared between animals of older ages versus younger ages.</p>
<p>You can read more on this Wiki page: <a href="https://en.wikipedia.org/wiki/Terminal_investment_hypothesis" class="uri">https://en.wikipedia.org/wiki/Terminal_investment_hypothesis</a>. We now actually completed a meta-analysis on this topic and published our findings - you can find our published paper at <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10176797/" class="uri">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10176797/</a>).</p>
<p>We hope the topic is quite appealing and quiet easy to understand.There are several published reviews on terminal investment hypothesis and we can expect many publications related to this topic, as well as many researchers working on it. Is this so?</p>
<p>Today, we will try to run bibliometric analyses on the relevant sample of literature. Note that there are a few available R packages (and many other online/software tools, and new are being developed) that can perform some of the tasks which we will practice during this exercise, and often they can do much more.</p>
<p>For your own project you may want to try to use some other tools, but there is no single “perfect” tool that fits all possible analyses and that is easy and usable for all disciplines and types of research questions. Also note that the main purpose of this exercise is to familiarize you with the basic principles/issues of bibliometric analyses. You can always learn more in your own time if you are interested.</p>
<hr>
<p><em>The search</em></p>
<p>First, we need to find a representative sample of academic publications on our topic of choice. For this, we will use cross-disciplinary database of academic literature, Scopus (<a href="https://www.scopus.com" class="uri">https://www.scopus.com</a>; owned by Elsevier). This database has the largest coverage of the published literature and should give us the most complete picture. An alternative database, commonly used for broad academic literature searches and analyses, is Web of Science: <a href="https://www.webofknowledge.com/" class="uri">https://www.webofknowledge.com/</a> (WoS; it has slightly different way of formatting search strings).</p>
<p>You should have free access to Scopus and WoS database on campus, but you may not be able to access it from outside the campus unless you use UNSW VPN or other university proxy servers (try logging in via the UNSW Library page, <a href="https://www.library.unsw.edu.au" class="uri">https://www.library.unsw.edu.au</a>, then search for a link to Scopus database).</p>
<p>Free-access broad-coverage databases include BASE (Bielfield Academic Search Engine, <a href="https://www.base-search.net/" class="uri">https://www.base-search.net/</a>) and Lens (https://www.lens.org). Google Scholar (<a href="https://scholar.google.com/" class="uri">https://scholar.google.com/</a>) is not recommended for this type of work, mainly because it is hard to export bibliometric records from the results list.</p>
<hr>
<section id="task-1" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="task-1">TASK 1</h3>
<p>Go to Scopus search page: <a href="https://www.scopus.com/search" class="uri">https://www.scopus.com/search</a>. Careful, there are several different search engines in Scopus: 1. Authors (also a new Researcher discovery) - information about authors (and affiliations).<br>
2. Sources - information about journals (and other collections of documents).<br>
3. Documents - information about journal articles (and other documents, such as book chapters) - WE WILL USE THIS ONE! It has two interfaces: Basic and Advanced.</p>
<p>In the Document basic search (a simple single search window), enter the phrase “terminal investment” (without quotation marks) in the basic search window, as follows:</p>
<div class="cell column-page quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-Scopus" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/Scopus_TI_res_simple_search.png" class="img-fluid figure-img" width="1264"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;1: <strong>?(caption)</strong></figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
<hr>
<p>Hey, this does not look good… - many documents were found, and most of them are completely unrelated (e.g.&nbsp;on building shipping terminals).<br>
Why is that?</p>
<hr>
<p>This is because our search is too simple. It allows us only to find the papers that explicitly mention terminal AND investment words in their title, abstract or keywords (the AND operator is a default one and it gets inserted automatically by the search software). To find a more relevant set of bibliographic records for our analyses, we need a more sophisticated search string.</p>
<p>Additionally, we will narrow down our topic a little bit more and aim to find papers that use immune challenge approach in wild or semi-wild animal species (so, we try to exclude established lab model species such as mice and rats, domesticated animals such as dogs and pigs, and humans). Finding the best search string is a bit of an art, so we just provide you with this one to save time (make sure to copy the whole thing - you may need to scroll to the right to see the complete string):</p>
<pre><code>TITLE-ABS-KEY ( ( "terminal investment"  OR  "reproductive effort"  OR  "fecundity compensation"  OR  "reproductive compensation"  OR  "reproductive fitness"  OR  "reproductive investment"  OR  "reproductive success"  OR  "Life History Trade-Off*"  OR  "Phenotypic Plasticity" )  AND  ( "immune challeng*"  OR  "immunochalleng*"  OR  "infect*"  OR  lipopolysaccharide  OR  lps  OR  phytohemagglutinin  OR  pha  OR  "sheep red blood cells"  OR  srbc  OR  implant  OR  vaccin* ) )  AND NOT  TITLE-ABS-KEY ( load  OR  human  OR  people  OR  men  OR  women  OR  infant*  OR  rat  OR  rats  OR  mouse  OR  mice  OR  pig*  OR  pork  OR  beef  OR  cattle  OR  sheep  OR  lamb*  OR  chicken*  OR  calf*  OR  *horse* )  </code></pre>
<hr>
</section>
<section id="task-2" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="task-2">TASK 2</h3>
<p>You need to copy and paste the above search string into the Advanced Search tab of the Scopus search page:</p>
<div class="cell column-page quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-Scopus_TI_search" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/Scopus_TI_res_all_adv_search.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;2: <strong>?(caption)</strong></figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
<hr>
<p>Press “Search”” button. You should see results roughly like this:</p>
<hr>
<div class="cell column-page quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-Scopus_TI" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/Scopus_TI_res_all.png" class="img-fluid figure-img" width="1277"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;3: <strong>?(caption)</strong></figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
<hr>
<p>There should be over 1,400 records retrieved from the Scopus database. Some look relevant and many are not, but that is always the case, for a systematic review or a meta-analysis we would manually screen all and exclude irrelevant papers).</p>
<p>On the left side of the results window you can see simple filters: year, most common author names, subject areas, etc. You can explore the whole set roughly by using “Analyze search results” link above the table of the hits:</p>
<div class="cell column-page quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-Scopus_TI_analyses" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/Scopus_TI_res_analaysis.png" class="img-fluid figure-img" width="1272"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;4: <strong>?(caption)</strong></figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
<p>There are more graphs below, and you can expand and customise them a bit too - have a look!</p>
<hr>
</section>
<section id="task-3" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="task-3">TASK 3</h3>
<p>Next, we can export the bibliographic records for more detailed bibliometric analyses in R. To do so, close the Scopus analyses window and go back to the list of the records found. First, select all records by clicking box “All” in the left top of the list of references. Then click the “Export” link to the right.</p>
<p>A pop-up window with the export options will appear.</p>
<p>First, select the format of the export: we will uses .bib file (BibTex format of references, one of the standard ones).<br>
Second, select which fields have to be exported by clicking the boxes on top of each column (or as needed).<br>
For bibliometric analyses on the citations among papers, it is essential to tick the box next to “Include references” (i.e.&nbsp;data on the cited documents).</p>
<p>Note that, unfortunately, Scopus limits number of exported records to 2,000. For longer lists of records, you will need to split them in smaller chunks for the export and then merge into a single larger data set (not covered in this tutorial; WoS export limits are 1,000 records).</p>
<div class="cell column-page quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell-output-display quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<div id="fig-Scopus_export" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/Scopus_TI_res_all_export_bib.png" class="img-fluid figure-img" width="1216"></p>
<p></p><figcaption class="figure-caption">Figure&nbsp;5: <strong>?(caption)</strong></figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
<hr>
<p>Click “Export” button. A file named “Scopus” (with extension matching your export type file, e.g., bib) will be saved to your downloads folder.<br>
Note that when you export references with their reference lists included in the records, the resulting files are quite large (in our case around 16Mb).</p>
<p>For this exercise, you can reduce your data set by applying some additional filters - e.g.&nbsp;including only literature from the last 5 years.</p>
<p>In case you did not succeed exporting the files (or wish to work with exactly the same ones we used, or you cannot access Scopus), the files downloaded on 27/05/2019 are provided (note that a standard way is to store them inside a “/data” sub-directory).</p>
<hr>
</section>
<section id="task-4" class="level3">
<h3 class="anchored" data-anchor-id="task-4">TASK 4</h3>
<p>Create a new Rmarkdown file to save your code (you can do this within new RStudio project). Install and upload <em>bibliometrix</em> R package:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">"bibliometrix"</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>) <span class="do">### installs bibliometrix package and dependencies</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(bibliometrix)   <span class="co">#uploads the package</span></span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Note: output not displayed for this chunk</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Upload the file exported from Scopus (you can use the one provided) into RStudio (note that the file path you may need to use on your computer may be different, e.g., “H:/Users/z1234567/Downloads/scopus.bib”).<br>
Then, convert the data from that file into internal <em>bibliometrix</em> format.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>bib <span class="ot">&lt;-</span> <span class="fu">convert2df</span>(<span class="st">"data/scopus.bib"</span>, <span class="at">dbsource =</span> <span class="st">"scopus"</span>, <span class="at">format =</span> <span class="st">"bibtex"</span>) <span class="co"># Convert to a bibliometric data frame</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; Converting your scopus collection into a bibliographic dataframe</span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; Done!</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; Generating affiliation field tag AU_UN from C1:  Done!</span></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(bib)</span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;  [1] "AU"         "DE"         "ID"         "C1"         "CR"        </span></span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;  [6] "JI"         "AB"         "AR"         "coden"      "RP"        </span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [11] "DT"         "DI"         "BE"         "FU"         "BN"        </span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [16] "SN"         "SO"         "LA"         "TC"         "PN"        </span></span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [21] "page_count" "PP"         "PU"         "PM"         "DB"        </span></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [26] "TI"         "url"        "VL"         "PY"         "FX"        </span></span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [31] "J9"         "AU_UN"      "AU1_UN"     "AU_UN_NR"   "SR_FULL"   </span></span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [36] "SR"</span></span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a><span class="co">#write.csv(bib, "data/bib_as_df.csv", row.names = FALSE) #if you want to save this data frame as a csv file</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>After some processing, an object called “bib” is created. It contains a data frame with each row corresponding to one exported publication from Scopus and with each column corresponding to a field exported from Scopus online database. If you tried to achieve this by exporting a csv file directly from Scopus, you would sometimes get a messy data frame, due to missing field values shifting the cells between columns.</p>
<hr>
</section>
<section id="task-5" class="level3">
<h3 class="anchored" data-anchor-id="task-5">TASK 5</h3>
<p>What are the contents of the columns of our “bib” data frame? Columns are labelled with a two-letter tags: AU, DE, ID, C1, CR, JI, AB, AR, coden, RP, DT, DI, BE, FU, BN, SN, SO, LA, TC, PN, page_count, PP, PU, PM, DB, TI, url, VL, PY, FX, J9, AU_UN, AU1_UN, AU_UN_NR, SR_FULL, SR.<br>
For a complete list and descriptions of field tags used in <em>bibliometrix</em> you can have a look at this file: <a href="http://www.bibliometrix.org/documents/Field_Tags_bibliometrix.pdf" class="uri">http://www.bibliometrix.org/documents/Field_Tags_bibliometrix.pdf</a>.<br>
Our data frame contains just a subset of these codes. Which ones?</p>
<p>Note that column <strong>bib$AU</strong> contains authors of each paper (as surnames and initials) separated by semicolon (;). We can easily split these strings and extract a list of all author names to a vector:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co"># head(bib$AU) #have a look at the few few records on your screen</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>authors <span class="ot">&lt;-</span> bib<span class="sc">$</span>AU</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>authors <span class="ot">&lt;-</span> <span class="fu">unlist</span>(<span class="fu">strsplit</span>(authors, <span class="st">";"</span>)) <span class="co">#split the records into individual authors</span></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>authors <span class="ot">&lt;-</span> authors[<span class="fu">order</span>(authors)] <span class="co">#order alphabetically</span></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(authors) <span class="co">#have a look again</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [1] "ABBOTT J"       "ABE A"          "ABEDON ST"      "ABO-SHEHADA M" </span></span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [5] "ABOUL-SOUD MAM" "ABRANTES N"</span></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a><span class="co"># View(unique(authors)) #use to see all the values</span></span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a><span class="co"># write.csv(authors, "data/author_list_uncleaned.csv", row.names = FALSE) #if you want to save this data frame as a csv file</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<hr>
</section>
<section id="task-6" class="level3">
<h3 class="anchored" data-anchor-id="task-6">TASK 6</h3>
<p>Cited references for each included paper are in the CR column of the “bib” data frame. They are in a single string, also separated by semicolon (;). We can have a look at them and check whether familiar names were cited, e.g.:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">dim</span>(bib) <span class="co">#dimensions of the data frame</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [1] 1167   36</span></span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(bib) <span class="co">#names of the columns of the data frame</span></span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;  [1] "AU"         "DE"         "ID"         "C1"         "CR"        </span></span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;  [6] "JI"         "AB"         "AR"         "coden"      "RP"        </span></span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [11] "DT"         "DI"         "BE"         "FU"         "BN"        </span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [16] "SN"         "SO"         "LA"         "TC"         "PN"        </span></span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [21] "page_count" "PP"         "PU"         "PM"         "DB"        </span></span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [26] "TI"         "url"        "VL"         "PY"         "FX"        </span></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [31] "J9"         "AU_UN"      "AU1_UN"     "AU_UN_NR"   "SR_FULL"   </span></span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [36] "SR"</span></span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a><span class="co">#bib$CR[1] #display a list of cited references for the first paper in the data frame </span></span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a><span class="co">#(we are not displaying it in this document as it is a very long string! - examine it on your screen instead)</span></span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a><span class="co">#look whether some of these names are cited:</span></span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a><span class="fu">grep</span>(<span class="st">"NAKAGAWA, S."</span>, bib<span class="sc">$</span>CR) </span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;  [1]   2   6   7  20  33  36  37  56  72  75 102 109 121 145 152 166 207 222 249</span></span>
<span id="cb5-17"><a href="#cb5-17" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [20] 285 293 312 330 361 362 368 370 401 440 455 471 475 489 501 512 560 562 573</span></span>
<span id="cb5-18"><a href="#cb5-18" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [39] 590 620 655 690 713 730 770</span></span>
<span id="cb5-19"><a href="#cb5-19" aria-hidden="true" tabindex="-1"></a><span class="fu">grep</span>(<span class="st">"CORNWELL, W."</span>, bib<span class="sc">$</span>CR) </span>
<span id="cb5-20"><a href="#cb5-20" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; [1] 15</span></span>
<span id="cb5-21"><a href="#cb5-21" aria-hidden="true" tabindex="-1"></a>bib[<span class="fu">grep</span>(<span class="st">"CORNWELL, W."</span>, bib<span class="sc">$</span>CR), <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>)] <span class="co">#who is citing?</span></span>
<span id="cb5-22"><a href="#cb5-22" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;                                                                                                           AU</span></span>
<span id="cb5-23"><a href="#cb5-23" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; MULETZ-WOLZ CR, 2019, J EVOL BIOL MULETZ-WOLZ CR;BARNETT SE;DIRENZO GV;ZAMUDIO KR;TOLEDO LF;JAMES TY;LIPS KR</span></span>
<span id="cb5-24"><a href="#cb5-24" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;                                                                                                                                                                                 DE</span></span>
<span id="cb5-25"><a href="#cb5-25" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; MULETZ-WOLZ CR, 2019, J EVOL BIOL AMPHIBIANS;  BATRACHOCHYTRIUM DENDROBATIDIS;  CHYTRID;  CLIMATE;  DISEASE ECOLOGY;  PATHOGEN;  PHENOTYPIC PLASTICITY;  PHYLOGENETIC CONSERVATISM</span></span>
<span id="cb5-26"><a href="#cb5-26" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;                                                                                  ID</span></span>
<span id="cb5-27"><a href="#cb5-27" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; MULETZ-WOLZ CR, 2019, J EVOL BIOL AMPHIBIA;  BATRACHOCHYTRIUM DENDROBATIDIS;  FUNGI</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<hr>
</section>
</section>
<section id="summarising-bibliometric-data" class="level2">
<h2 class="anchored" data-anchor-id="summarising-bibliometric-data">Summarising bibliometric data</h2>
<section id="task-7" class="level3">
<h3 class="anchored" data-anchor-id="task-7">TASK 7</h3>
<p>Luckily, <em>bibliometrix</em> package has a handy function that summarises the information contained in the “bib” data frame, so we can get some quick facts about our set of papers.</p>
<p><strong>Note:</strong> this and the following tasks require quite a bit of computational power, they may be slow or even halt on your computer.<br>
In such case, for this exercise, make your data frame smaller by sub-setting it, e.g.:<br>
“bib &lt;- bib[1:500, ] #taking first 500 records”. However, the results and plots you will produce with a subsetted data frame will differ from the ones presented in this document.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Preliminary descriptive analyses </span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>results <span class="ot">&lt;-</span> <span class="fu">biblioAnalysis</span>(bib, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="co">#summary(object = results, k = 10, pause = TRUE) #display a series of summary tables</span></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Reduce the data set to the subset first 500 records</span></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>bib <span class="ot">&lt;-</span> bib[<span class="dv">1</span><span class="sc">:</span><span class="dv">500</span>, ] <span class="co">#taking first 500 records</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a><span class="co">#summary(object = results, k = 10, pause = TRUE) #display a series of summary tables</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Using <em>summary</em> function on <em>bibliometrix</em> results, you get several screens with various tables summarising bibliometric data from our data frame - how many documents, journals, keywords, authors, publications time-span, collaboration index, annual publication growth rate, most prolific authors, publications per country, per journal, per keywords, etc.</p>
<p>You can automatically plot some of these tables (hit “return”” to display next graph, and later you can use arrows in the top left of the plots pane to move back and forth between consecutive plots saved in the RStudio memory):</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(results, <span class="at">k =</span> <span class="dv">10</span>, <span class="at">pause=</span><span class="cn">TRUE</span>) <span class="co">#this takes top 10 values from each plottable table</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; Hit &lt;Return&gt; to see next plot:</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid" width="672"></p>
</div>
<pre><code>#&gt; Hit &lt;Return&gt; to see next plot:</code></pre>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-11-2.png" class="img-fluid" width="672"></p>
</div>
<pre><code>#&gt; Hit &lt;Return&gt; to see next plot:</code></pre>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-11-3.png" class="img-fluid" width="672"></p>
</div>
<pre><code>#&gt; Hit &lt;Return&gt; to see next plot:</code></pre>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-11-4.png" class="img-fluid" width="672"></p>
</div>
<pre><code>#&gt; Hit &lt;Return&gt; to see next plot:</code></pre>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-11-5.png" class="img-fluid" width="672"></p>
</div>
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a><span class="co">#the code below is for saving these plots into a pdf:</span></span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a><span class="co"># pdf(file = "plots/bib_descriptive_plots.pdf", height = 8, width = 8, pointsize=10) #</span></span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a><span class="co"># plot(results, k = 20, pause=FALSE) #this takes top 20 values from each plottable table</span></span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a><span class="co"># dev.off()</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<hr>
</section>
<section id="task-8" class="level3">
<h3 class="anchored" data-anchor-id="task-8">TASK 8</h3>
<p>The cited papers from the CR field of the data frame can be analysed using function <em>citations</em>.<br>
Function <em>citations</em> makes it easy to generate the frequency tables of the most cited papers or the most cited first authors.</p>
<p>Ten most cited papers:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a>mostcitedP <span class="ot">&lt;-</span> <span class="fu">citations</span>(bib, <span class="at">field =</span> <span class="st">"article"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a><span class="fu">cbind</span>(mostcitedP<span class="sc">$</span>Cited[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>]) <span class="co">#first 10</span></span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;                                                                                                                                                                                                       [,1]</span></span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; LOCHMILLER, R.L., DEERENBERG, C., TRADE-OFFS IN EVOLUTIONARY IMMUNOLOGY: JUST WHAT IS THE COST OF IMMUNITY? (2000) OIKOS, 88, PP. 87-98                                                                 21</span></span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; MORET, Y., SCHMID-HEMPEL, P., SURVIVAL FOR IMMUNITY: THE PRICE OF IMMUNE SYSTEM ACTIVATION FOR BUMBLEBEE WORKERS (2000) SCIENCE, 290, PP. 1166-1168                                                     10</span></span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; ANDERSON, R.M., MAY, R.M., COEVOLUTION OF HOSTS AND PARASITES (1982) PARASITOLOGY, 85, PP. 411-426                                                                                                       9</span></span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; ROLFF, J., SIVA-JOTHY, M.T., INVERTEBRATE ECOLOGICAL IMMUNOLOGY (2003) SCIENCE, 301, PP. 472-475                                                                                                         8</span></span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; CLUTTON-BROCK, T.H., REPRODUCTIVE EFFORT AND TERMINAL INVESTMENT IN ITEROPAROUS ANIMALS (1984) AM. NAT., 123, PP. 212-229                                                                                7</span></span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; MINCHELLA, D.J., LOVERDE, P.T., A COST OF INCREASED EARLY REPRODUCTIVE EFFORT IN THE SNAIL BIOMPHALARIA GLABRATA (1981) AM. NAT., 118, PP. 876-881                                                       7</span></span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; SHELDON, B.C., VERHULST, S., ECOLOGICAL IMMUNOLOGY: COSTLY PARASITE DEFENCES AND TRADE-OFFS IN EVOLUTIONARY ECOLOGY (1996) TRENDS IN ECOLOGY &amp; EVOLUTION, 11, PP. 317-321                                7</span></span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; AHMED, A.M., BAGGOTT, S.L., MAINGON, R., HURD, H., THE COSTS OF MOUNTING AN IMMUNE RESPONSE ARE REFLECTED IN THE REPRODUCTIVE FITNESS OF THE MOSQUITO ANOPHELES GAMBIAE (2002) OIKOS, 97, PP. 371-377    6</span></span>
<span id="cb13-12"><a href="#cb13-12" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; FORBES, M.R.L., PARASITISM AND HOST REPRODUCTIVE EFFORT (1993) OIKOS, 67, PP. 444-450                                                                                                                    6</span></span>
<span id="cb13-13"><a href="#cb13-13" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; HAMILTON, W.D., ZUK, M., HERITABLE TRUE FITNESS AND BRIGHT BIRDS: A ROLE FOR PARASITES? (1982) SCIENCE, 218, PP. 384-387                                                                                 6</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Ten most cited authors:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>mostcitedA <span class="ot">&lt;-</span> <span class="fu">citations</span>(bib, <span class="at">field =</span> <span class="st">"author"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a><span class="fu">cbind</span>(mostcitedA<span class="sc">$</span>Cited[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>]) <span class="co">#first 10</span></span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;                 [,1]</span></span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; POULIN R         181</span></span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; SCHMID HEMPEL P  181</span></span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; HASSELQUIST D    160</span></span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; WINGFIELD J C    147</span></span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; SHELDON B C      117</span></span>
<span id="cb14-9"><a href="#cb14-9" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; HOFFMANN A A     116</span></span>
<span id="cb14-10"><a href="#cb14-10" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; MØLLER A P       114</span></span>
<span id="cb14-11"><a href="#cb14-11" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; READ A F         113</span></span>
<span id="cb14-12"><a href="#cb14-12" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; RANTALA M J      106</span></span>
<span id="cb14-13"><a href="#cb14-13" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; BENSCH S         104</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Function <em>localCitations</em> generates the frequency table of the <strong>locally</strong> most cited authors. Locally means that citations are counted only within the given data set - i.e.&nbsp;how many times an author/paper that is in this data set has been cited by other authors/papers also in the data set.</p>
<p>Ten most frequent local cited authors and papers:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>mostcitedLA <span class="ot">&lt;-</span> <span class="fu">localCitations</span>(bib, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>mostcitedLA<span class="sc">$</span>Authors[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>,]</span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;               Author LocalCitations</span></span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 1169   LÓPEZ-GÓMEZ M              9</span></span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 1928 VERDEJO-LUCAS S              9</span></span>
<span id="cb15-6"><a href="#cb15-6" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 165       BILLINGS G              8</span></span>
<span id="cb15-7"><a href="#cb15-7" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 435         DAVIS BM              8</span></span>
<span id="cb15-8"><a href="#cb15-8" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 879       HUBBARD TP              8</span></span>
<span id="cb15-9"><a href="#cb15-9" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 1124        LEUNG KY              8</span></span>
<span id="cb15-10"><a href="#cb15-10" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 1158           LIU Q              8</span></span>
<span id="cb15-11"><a href="#cb15-11" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 1439         PARK JS              8</span></span>
<span id="cb15-12"><a href="#cb15-12" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 1961       WALDOR MK              8</span></span>
<span id="cb15-13"><a href="#cb15-13" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 1974          WANG Q              8</span></span>
<span id="cb15-14"><a href="#cb15-14" aria-hidden="true" tabindex="-1"></a>mostcitedLA<span class="sc">$</span>Papers[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>,]</span>
<span id="cb15-15"><a href="#cb15-15" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;                                    Paper                                DOI</span></span>
<span id="cb15-16"><a href="#cb15-16" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 323              ASGHAR M, 2015, SCIENCE            10.1126/science.1261121</span></span>
<span id="cb15-17"><a href="#cb15-17" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 448           LEVENTHAL GE, 2014, AM NAT                     10.1086/675242</span></span>
<span id="cb15-18"><a href="#cb15-18" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 312              BOWERS EK, 2015, AM NAT                     10.1086/681017</span></span>
<span id="cb15-19"><a href="#cb15-19" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 254   SCHWENKE RA, 2016, ANN REV ENTOMOL 10.1146/annurev-ento-010715-023924</span></span>
<span id="cb15-20"><a href="#cb15-20" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 440       PODMOKŁA E, 2014, J AVIAN BIOL   10.1111/j.1600-048X.2013.00284.x</span></span>
<span id="cb15-21"><a href="#cb15-21" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 500 GONZÁLEZ-TOKMAN DM, 2013, FUNCT ECOL            10.1111/1365-2435.12072</span></span>
<span id="cb15-22"><a href="#cb15-22" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 320          VÉZILIER J, 2015, BIOL LETT             10.1098/rsbl.2014.0840</span></span>
<span id="cb15-23"><a href="#cb15-23" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 192        GIEHR J, 2017, R SOC OPEN SCI                10.1098/rsos.170547</span></span>
<span id="cb15-24"><a href="#cb15-24" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 236        BRANNELLY LA, 2016, OPEN BIOL                10.1098/rsob.150251</span></span>
<span id="cb15-25"><a href="#cb15-25" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 273         CROSSIN GT, 2016, FUNCT ECOL            10.1111/1365-2435.12482</span></span>
<span id="cb15-26"><a href="#cb15-26" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt;     Year LCS GCS</span></span>
<span id="cb15-27"><a href="#cb15-27" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 323 2015   9 166</span></span>
<span id="cb15-28"><a href="#cb15-28" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 448 2014   8  15</span></span>
<span id="cb15-29"><a href="#cb15-29" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 312 2015   6  30</span></span>
<span id="cb15-30"><a href="#cb15-30" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 254 2016   5  67</span></span>
<span id="cb15-31"><a href="#cb15-31" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 440 2014   5  20</span></span>
<span id="cb15-32"><a href="#cb15-32" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 500 2013   5  17</span></span>
<span id="cb15-33"><a href="#cb15-33" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 320 2015   4  14</span></span>
<span id="cb15-34"><a href="#cb15-34" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 192 2017   3   9</span></span>
<span id="cb15-35"><a href="#cb15-35" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 236 2016   3  18</span></span>
<span id="cb15-36"><a href="#cb15-36" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; 273 2016   3  44</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<hr>
</section>
</section>
<section id="creating-bibliometric-networks" class="level2">
<h2 class="anchored" data-anchor-id="creating-bibliometric-networks">Creating bibliometric networks</h2>
<p>So far, we looked only at the numbers - who or what gets cited most, either from the main papers list or from the lists of the references within these papers. Now it is time to look at the actual networks of citations and also other types of networks that can be created using our data set.</p>
<p>To do so, we will be creating various rectangular matrices which reflect connections of different attributes of Documents/Authors. These matrices than can be plotted as bipartite networks and analysed.</p>
<p>Co-citation or coupling networks are a special type of networks build from the information on scientific papers containing references to other scientific papers.</p>
<p>Package <em>bibliometrix</em> contains function <em>biblioNetwork</em> which makes creating bibliographic networks easy. This function can create the most frequently used coupling networks: Authors, Sources, and Countries.</p>
<hr>
<section id="task-9" class="level3">
<h3 class="anchored" data-anchor-id="task-9">TASK 9</h3>
<p><strong>Bibliographic coupling</strong> - two articles are bibliographically coupled if they share at least one reference from their reference lists, i.e.&nbsp;at least one cited source appears in the reference lists/bibliographies of both papers (Kessler, 1963).</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"coupling"</span>, <span class="at">network =</span> <span class="st">"references"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>net <span class="ot">=</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted =</span> <span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Papers' bibliographic coupling"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-15-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Above, we plotted only the top 10 most coupled papers (n = 10). Try increasing this number to 100 (would not recommend further increasing the number of displayed nodes - it gets slow and messy).<br>
What happens and why?</p>
<hr>
</section>
<section id="task-10" class="level3">
<h3 class="anchored" data-anchor-id="task-10">TASK 10</h3>
<p><strong>Author’s bibliographic coupling</strong> - two authors are bibliographically coupled if they share at least one reference form their reference lists.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"coupling"</span>, <span class="at">network =</span> <span class="st">"authors"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>net <span class="ot">=</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted =</span> <span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Authors' bibliographic coupling"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.8</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Above, we plotted only the top 10 most coupled authors (n = 10). Try increasing this number to 100 (would not recommend further increasing the number of displayed nodes to &gt;50 - it gets slow and messy).<br>
What happens and why?</p>
<hr>
</section>
<section id="task-11" class="level3">
<h3 class="anchored" data-anchor-id="task-11">TASK 11</h3>
<p><strong>Bibliographic co-citation</strong> is kind of opposite to bibliographic coupling, in so that two papers are linked by co-citation when both are cited in a third paper.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib[<span class="dv">1</span><span class="sc">:</span><span class="dv">50</span>,], <span class="at">analysis =</span> <span class="st">"co-citation"</span>, <span class="at">network =</span> <span class="st">"references"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted=</span><span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Papers' co-citations"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-17-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Note that for creating this matrix we only used first 50 papers from our data set - this is because the resulting matrix is a matrix of ALL cited papers and it gets HUGE). Also, we plotted only the top 10 most coupled papers (n=10). Try increasing this number to 20 (would not recommend further increasing the number of displayed nodes to &gt;50 - it gets slow and messy).<br>
What happens and why?</p>
<hr>
</section>
<section id="task-12" class="level3">
<h3 class="anchored" data-anchor-id="task-12">TASK 12</h3>
<p><strong>Bibliographic collaboration</strong> is a network where nodes are authors and links are co-authorships of the papers.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"collaboration"</span>, <span class="at">network =</span> <span class="st">"authors"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted =</span> <span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Authors' collaborations"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-18-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Above, we plotted only the top 10 most collaborating authors (n = 10). Try increasing this number to 100 (would not recommend further increasing the number of displayed nodes - it gets slow and messy).<br>
What happens and why?</p>
<hr>
</section>
<section id="task-13" class="level3">
<h3 class="anchored" data-anchor-id="task-13">TASK 13</h3>
<p><strong>Country Scientific Collaboration</strong> - we can visualise authors from which countries publish papers together most frequently.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>bib <span class="ot">&lt;-</span> <span class="fu">metaTagExtraction</span>(bib, <span class="at">Field =</span> <span class="st">"AU_CO"</span>, <span class="at">sep =</span> <span class="st">";"</span>) <span class="co">#we need to extract countries from the affiliations first</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"collaboration"</span>, <span class="at">network =</span> <span class="st">"countries"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Country Collaboration"</span>, <span class="at">type =</span> <span class="st">"auto"</span>, <span class="at">size =</span> <span class="cn">TRUE</span>, <span class="at">remove.multiple =</span> <span class="cn">FALSE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-19-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Above, we plotted only the top 10 most collaborating countries (n = 10). Try increasing this number to 50 (would not recommend further increasing the number of displayed nodes to &gt;100 - it gets slow and messy).<br>
What happens and why?</p>
<hr>
</section>
<section id="task-14" class="level3">
<h3 class="anchored" data-anchor-id="task-14">TASK 14</h3>
<p><strong>Keyword co-occurrences</strong> - we can also visualise which papers share most keywords (from the Scopus database).</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"co-occurrences"</span>, <span class="at">network =</span> <span class="st">"keywords"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">n =</span> <span class="dv">50</span>, <span class="at">Title =</span> <span class="st">"Keyword co-occurance"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> T, <span class="at">remove.multiple =</span> <span class="cn">FALSE</span>, <span class="at">labelsize =</span> <span class="fl">0.7</span>, <span class="at">edgesize =</span> <span class="dv">5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Try replacing network = “keywords” with network = “author_keywords” and see what happens. You can also try to display fewer/more keywords in the plot.</p>
<hr>
</section>
<section id="task-15" class="level3">
<h3 class="anchored" data-anchor-id="task-15">TASK 15</h3>
<p><strong>Note</strong>: you may want to skip this step on a big data set or a slow computer.</p>
<p><strong>Co-Word Analysis</strong> - uses the word co-occurrences in a bibliographic collection to map the conceptual structure of research. It works via a separate function <em>conceptualStructure</em> that creates a conceptual structure map of a scientific field performing Correspondence Analysis (CA), Multiple Correspondence Analysis (MCA) or Metric Multidimensional Scaling (MDS) and Clustering of a bipartite network of terms extracted from keyword, title or abstract fields of the data frame. Note that three different graphs are created as an output.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>CS <span class="ot">&lt;-</span> <span class="fu">conceptualStructure</span>(bib, <span class="at">field =</span> <span class="st">"ID"</span>, <span class="at">minDegree =</span> <span class="dv">20</span>, <span class="at">k.max =</span> <span class="dv">5</span>, <span class="at">stemming =</span> <span class="cn">FALSE</span>, <span class="at">labelsize =</span> <span class="dv">10</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-21-1.png" class="img-fluid" width="672"></p>
</div>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-21-2.png" class="img-fluid" width="672"></p>
</div>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-21-3.png" class="img-fluid" width="672"></p>
</div>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-21-4.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>The code above uses field = “ID”. ID stands for “Keywords Plus associated by ISI or SCOPUS database”, to analyse the “conceptualStructure”. You could try using authors keywords, “DE” field, instead of “ID”.<br>
Is the new map different?</p>
<hr>
</section>
<section id="task-16" class="level3">
<h3 class="anchored" data-anchor-id="task-16">TASK 16</h3>
<p><strong>Note</strong>: you may want to skip this step on a big data set or a slow computer.<br>
<strong>Historical Direct Citation Network</strong> - represents a chronological network map of most relevant direct citations in a bibliographic collection, i.e who is citing whom and in what order. <em>histNetwork</em> function calculates a chronological direct citation network matrix which then is plotted using <em>histPlot</em>:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="co">#options(width = 130)</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>histResults <span class="ot">&lt;-</span> <span class="fu">histNetwork</span>(bib, <span class="at">min.citations =</span> <span class="dv">10</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; SCOPUS DB: Searching local citations (LCS) by document titles (TI) and DOIs...</span></span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; </span></span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a><span class="co">#&gt; Found 59 documents with no empty Local Citations (LCS)</span></span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">histPlot</span>(histResults, <span class="at">labelsize =</span> <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<p><img src="learn_bibliometrix_files/figure-html/unnamed-chunk-22-1.png" class="img-fluid" width="672"></p>
</div>
<pre><code>#&gt; 
#&gt;  Legend
#&gt; 
#&gt;                                      Label
#&gt; 1            BRANNELLY LA, 2016, OPEN BIOL
#&gt; 2           DUFFIELD KR, 2015, J EVOL BIOL
#&gt; 3  NYSTRAND M, 2014, PROC R SOC B BIOL SCI
#&gt; 4         MARTINEZ AJ, 2014, BMC EVOL BIOL
#&gt; 5        PIGEAULT R, 2018, INT J PARASITOL
#&gt; 6             DUFFIELD KR, 2018, EVOLUTION
#&gt; 7            GIEHR J, 2017, R SOC OPEN SCI
#&gt; 8           DUFFIELD KR, 2019, J EVOL BIOL
#&gt; 9                 LUU H, 2017, J EVOL BIOL
#&gt; 10         NYSTRAND M, 2016, BMC EVOL BIOL
#&gt; 11              DENNIS AB, 2017, EVOLUTION
#&gt;                                                                                                                                                    Author_Keywords
#&gt; 1                                                             CHYTRID FUNGUS;  OOGENESIS;  REPRODUCTION;  SPERMATOGENESIS;  TERMINAL INVESTMENT;  WILDLIFE DISEASE
#&gt; 2                                                             LIFE-HISTORY THEORY;  NUPTIAL FOOD GIFT;  REPRODUCTIVE EFFORT;  SPERMATOPHYLAX;  TERMINAL INVESTMENT
#&gt; 3                                                    HEAT-KILLED BACTERIA;  IMMUNE ELICITOR;  IMMUNE TRANSFER;  LIFE HISTORY;  MATERNAL EFFECTS;  PATERNAL EFFECTS
#&gt; 4                                                                                             FECUNDITY;  IMMUNITY;  RESISTANCE;  SELECTION;  SYMBIONT;  TRADE-OFF
#&gt; 5                                                        CO-INFECTION;  HAEMOPROTEUS;  LEUCOCYTOZOON;  LIFE-HISTORY TRAITS;  PARUS MAJOR;  PLASMODIUM;  TRADE-OFFS
#&gt; 6                                     AGE-DEPENDENT REPRODUCTIVE EFFORT;  FECUNDITY COMPENSATION;  GRYLLODES SIGILLATUS;  RESIDUAL REPRODUCTIVE VALUE;  TRADE-OFFS
#&gt; 7                                                                                                   INFECTION;  REPRODUCTION;  SOCIAL INSECT;  TERMINAL INVESTMENT
#&gt; 8                                                 FECUNDITY COMPENSATION;  GENOTYPE-BY-ENVIRONMENT INTERACTION;  LIFE-HISTORY THEORY;  RESIDUAL REPRODUCTIVE VALUE
#&gt; 9              BET-HEDGING;  FECUNDITY COMPENSATION;  IMMUNE PRIMING;  LIFE-HISTORY TRADE-OFFS;  PHENOTYPIC PLASTICITY;  SOMATIC MAINTENANCE;  TERMINAL INVESTMENT
#&gt; 10 ANTICIPATORY EFFECTS;  MATERNAL EFFECTS;  MATERNAL STRESS;  PARENTAL EFFECTS;  PHENOTYPIC PLASTICITY;  TRANSGENERATIONAL EFFECTS;  TRANSGENERATIONAL PLASTICITY
#&gt; 11                                                                ADAPTATION;  EXPERIMENTAL EVOLUTION;  GENE EXPRESSION;  LYSIPHLEBUS FABARUM;  PARASITOID;  VENOM
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                KeywordsPlus
#&gt; 1                                                                                                                                                                                                                                                                                                                                                                                                                                                ANIMAL;  ANURA;  CHYTRIDIOMYCETES;  COMMUNICABLE DISEASE;  ENDANGERED SPECIES;  FEMALE;  GAMETOGENESIS;  MALE;  MICROBIOLOGY;  MYCOSIS;  ORGAN SIZE;  PATHOGENICITY;  PHYSIOLOGY;  POPULATION DYNAMICS;  VETERINARY; ANIMALS;  ANURA;  CHYTRIDIOMYCOTA;  COMMUNICABLE DISEASES; EMERGING;  ENDANGERED SPECIES;  FEMALE;  GAMETOGENESIS;  MALE;  MYCOSES;  ORGAN SIZE;  POPULATION DYNAMICS
#&gt; 2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            AMINO ACID;  BACTERIUM;  CRICKET;  FITNESS;  IMMUNE RESPONSE;  LIFE EXPECTANCY;  LIFE HISTORY THEORY;  MORTALITY;  NUPTIAL GIFT;  REPRODUCTIVE COST;  REPRODUCTIVE EFFORT;  SPERMATOPHORE;  SURVIVAL;  TASTE;  TRADE-OFF; GRYLLIDAE;  GRYLLODES SIGILLATUS; ANIMAL;  GRYLLIDAE;  MALE;  PHYSIOLOGY;  TASTE; ANIMALS;  GRYLLIDAE;  MALE;  TASTE
#&gt; 3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         FEMALE BEHAVIOR;  FLY;  GENE EXPRESSION;  GENOTYPE;  IMMUNE SYSTEM;  LIFE HISTORY;  MATERNAL EFFECT;  PATERNAL EFFECT;  PHENOTYPE;  REPRODUCTIVE SUCCESS; DROSOPHILA MELANOGASTER
#&gt; 4                                                                                                                                                                                                                   APHID;  FECUNDITY;  HERITABILITY;  IMMUNITY;  LONGEVITY;  NATURAL ENEMY;  PARASITOID;  SYMBIONT;  TRADE-OFF;  WASP; ACYRTHOSIPHON PISUM;  ANIMALIA;  APHIDIDAE;  APHIDIUS ERVI;  BACTERIA (MICROORGANISMS);  CANDIDATUS HAMILTONELLA DEFENSA;  VESPOIDEA; ANIMAL;  APHID;  ARTICLE;  BACTERIOPHAGE;  EVOLUTION;  FERTILITY;  GAMMAPROTEOBACTERIA;  IMMUNOLOGY;  LONGEVITY;  MICROBIOLOGY;  PARASITOLOGY;  PHYSIOLOGY;  REPRODUCTIVE FITNESS;  SYMBIOSIS;  VIROLOGY;  WASP; ANIMALS;  APHIDS;  BACTERIOPHAGES;  BIOLOGICAL EVOLUTION;  FERTILITY;  GAMMAPROTEOBACTERIA;  GENETIC FITNESS;  LONGEVITY;  SYMBIOSIS;  WASPS
#&gt; 5                                                                                                                                                                                                                                                                                           HOST-PARASITE INTERACTION;  INFECTIVITY;  LIFE HISTORY TRAIT;  PARASITIC DISEASE;  PASSERINE;  REPRODUCTION;  SURVIVAL;  TRADE-OFF;  WILD POPULATION; ADULT;  ARTICLE;  BLOOD SAMPLING;  BODY CONSTITUTION;  CONTROLLED STUDY;  EVIDENCE BASED PRACTICE;  FEMALE;  HAEMOSPORIDA;  MALE;  MARK RECAPTURE;  MIXED INFECTION;  MOLECULAR DIAGNOSIS;  NONHUMAN;  PARASITE VIRULENCE;  PARASITOSIS;  PARUS MAJOR;  PLASMODIUM;  POPULATION RESEARCH;  REPRODUCTION;  SURVIVAL RATE;  WILD SPECIES; AVES;  HAEMOPROTEUS;  LEUCOCYTOZOON;  PARUS MAJOR
#&gt; 6                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          AGE STRUCTURE;  ANTIMICROBIAL ACTIVITY;  BACTERIUM;  CRICKET;  FECUNDITY;  INFECTIOUS DISEASE;  REPRODUCTION;  REPRODUCTIVE EFFORT;  TRADE-OFF; GRYLLIDAE;  GRYLLODES SIGILLATUS
#&gt; 7                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      &lt;NA&gt;
#&gt; 8                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      &lt;NA&gt;
#&gt; 9                                                                                                                                                                                                                                                                                                                                                                                                                                                         BET-HEDGING;  EVOLUTIONARY BIOLOGY;  FECUNDITY;  FITNESS;  IMMUNE SYSTEM;  LIFE HISTORY TRAIT;  MORTALITY;  PARASITE;  PARENTAL INVESTMENT;  PHENOTYPE;  REPRODUCTION;  REPRODUCTIVE STRATEGY;  SURVIVAL; ANIMAL;  HOST PARASITE INTERACTION;  LONGEVITY;  REPRODUCTION;  REPRODUCTIVE FITNESS; ANIMALS;  GENETIC FITNESS;  HOST-PARASITE INTERACTIONS;  LONGEVITY;  REPRODUCTION
#&gt; 10                                                                                                                                                                                                                                                                                                                            ADAPTIVE RADIATION;  BACTERIUM;  DESIGN;  ENVIRONMENTAL STRESS;  FLY;  HETEROGENEITY;  HYBRIDIZATION;  PATHOGEN;  PHENOTYPIC PLASTICITY;  REPRODUCTIVE SUCCESS;  SEXUAL DIMORPHISM; DROSOPHILA MELANOGASTER;  INVERTEBRATA; ADAPTATION;  ANIMAL;  DROSOPHILA MELANOGASTER;  ENVIRONMENT;  FEMALE;  MALE;  MICROBIOLOGY;  PHYSIOLOGICAL STRESS;  PHYSIOLOGY;  REPRODUCTION; ADAPTATION; PHYSIOLOGICAL;  ANIMALS;  DROSOPHILA MELANOGASTER;  ENVIRONMENT;  FEMALE;  MALE;  REPRODUCTION;  STRESS; PHYSIOLOGICAL
#&gt; 11 ADAPTATION;  APHID;  BEHAVIORAL RESPONSE;  COEVOLUTION;  ENDOSYMBIONT;  EVOLUTIONARY BIOLOGY;  EXPERIMENTAL STUDY;  GENE EXPRESSION;  GENETIC VARIATION;  HOST SPECIFICITY;  HOST-SYMBIONT INTERACTION;  NATURAL SELECTION;  PARASITOID;  PHENOTYPIC PLASTICITY;  VENOM;  WASP; APHIDIDAE;  APHIS FABAE;  CANDIDATUS HAMILTONELLA DEFENSA;  LYSIPHLEBUS FABARUM; TRANSCRIPTOME; ADAPTATION;  ANIMAL;  APHID;  EGG LAYING;  ENTEROBACTERIACEAE;  FEMALE;  GENE;  GENETICS;  HOST PARASITE INTERACTION;  LIFE HISTORY TRAIT;  MICROBIOLOGY;  MOLECULAR EVOLUTION;  PARASITOLOGY;  PATHOGENICITY;  SYMBIOSIS;  WASP; ADAPTATION; PHYSIOLOGICAL;  ANIMALS;  APHIDS;  ENTEROBACTERIACEAE;  EVOLUTION; MOLECULAR;  FEMALE;  GENES; INSECT;  HOST-PARASITE INTERACTIONS;  LIFE HISTORY TRAITS;  OVIPOSITION;  SYMBIOSIS;  TRANSCRIPTOME;  WASPS
#&gt;                             DOI Year LCS GCS
#&gt; 1           10.1098/rsob.150251 2016   3  18
#&gt; 2             10.1111/jeb.12703 2015   3  12
#&gt; 3        10.1098/rspb.2014.1242 2014   1  11
#&gt; 4      10.1186/1471-2148-14-127 2014   3  32
#&gt; 5  10.1016/j.ijpara.2018.06.007 2018   0   1
#&gt; 6             10.1111/evo.13443 2018   0   2
#&gt; 7           10.1098/rsos.170547 2017   0   9
#&gt; 8             10.1111/jeb.13478 2019   0   0
#&gt; 9             10.1111/jeb.13138 2017   0   2
#&gt; 10    10.1186/s12862-016-0737-6 2016   0   3
#&gt; 11            10.1111/evo.13333 2017   0   9</code></pre>
</div>
<p>Only articles with minimum of 10 citations are included in the above analysis, if you change this number to a higher value, the analyses will be quicker and the plot less dense. Have a try!</p>
<hr>
<p><strong>MORE TO DO</strong><br>
You can use different types of network plots - just tweak “type” parameter in the <em>networkPlot</em> function (check the vignette for the available options). “Type” parameter indicates the network map plotting layout: circle, kamada-kawai, mds, etc.</p>
<p>You can use non-R tools to visualise bibliographic networks, e.g.&nbsp;VOSviewer software by Nees Jan van Eck and Ludo Waltman (<a href="http://www.vosviewer.com" class="uri">http://www.vosviewer.com</a>). When in R function you use type=“vosviewer”, the function will export the network as a standard “pajek” network file (named “vosnetwork.net”), which can then be used in other network-plotting software, including VOSviewer.</p>
<hr>
</section>
</section>
<section id="resources" class="level2">
<h2 class="anchored" data-anchor-id="resources">Resources</h2>
<ul>
<li>Bibliometrix webpage (includes an online “biblioshiny” app): <a href="http://www.bibliometrix.org/" class="uri">http://www.bibliometrix.org/</a>.<br>
</li>
<li>A more detailed <em>bibliometrix</em> information and links on GitHub: <a href="https://github.com/massimoaria/bibliometrix" class="uri">https://github.com/massimoaria/bibliometrix</a>.<br>
</li>
<li>Another useful R package - Revtools: <a href="http://revtools.net/" class="uri">http://revtools.net/</a>.<br>
</li>
<li>Stand-alone GUI software (introduced above) - VOSviewer: <a href="http://www.vosviewer.com/" class="uri">http://www.vosviewer.com/</a>.</li>
</ul>
<!-- -->

</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb25" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Learning bibliometrix"</span></span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a><span class="an">format:</span></span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a><span class="co">  html:</span></span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a><span class="co">    code-fold: true</span></span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a><span class="co">    code-tools: true</span></span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a><span class="an">editor:</span><span class="co"> visual</span></span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a><span class="an">author:</span><span class="co"> "M.Lagisz"</span></span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a><span class="an">date:</span><span class="co"> 2023-05-31</span></span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a><span class="an">date-format:</span><span class="co"> short</span></span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a><span class="an">output:</span></span>
<span id="cb25-12"><a href="#cb25-12" aria-hidden="true" tabindex="-1"></a><span class="co">  pdf_document: default</span></span>
<span id="cb25-13"><a href="#cb25-13" aria-hidden="true" tabindex="-1"></a><span class="co">  html_document: default</span></span>
<span id="cb25-14"><a href="#cb25-14" aria-hidden="true" tabindex="-1"></a><span class="an">editor_options:</span><span class="co"> </span></span>
<span id="cb25-15"><a href="#cb25-15" aria-hidden="true" tabindex="-1"></a><span class="co">  chunk_output_type: console</span></span>
<span id="cb25-16"><a href="#cb25-16" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb25-17"><a href="#cb25-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-20"><a href="#cb25-20" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-21"><a href="#cb25-21" aria-hidden="true" tabindex="-1"></a><span class="co">#| include: false</span></span>
<span id="cb25-22"><a href="#cb25-22" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span>opts_chunk<span class="sc">$</span><span class="fu">set</span>(<span class="at">error =</span> <span class="cn">TRUE</span>) <span class="co">#allow some execution errors for demonstration purposes</span></span>
<span id="cb25-23"><a href="#cb25-23" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span>opts_chunk<span class="sc">$</span><span class="fu">set</span>(<span class="at">eval =</span> <span class="cn">TRUE</span>, <span class="at">echo =</span> <span class="cn">TRUE</span>, <span class="at">warning =</span> <span class="cn">FALSE</span>, <span class="at">collapse =</span> <span class="cn">TRUE</span>, <span class="at">comment =</span> <span class="st">"#&gt;"</span>)</span>
<span id="cb25-24"><a href="#cb25-24" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span>
<span id="cb25-25"><a href="#cb25-25" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(bibliometrix)   </span>
<span id="cb25-26"><a href="#cb25-26" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-27"><a href="#cb25-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-28"><a href="#cb25-28" aria-hidden="true" tabindex="-1"></a>**Before the lesson:**    </span>
<span id="cb25-29"><a href="#cb25-29" aria-hidden="true" tabindex="-1"></a>Please make sure you got the latest RStudio and latest R version installed.   </span>
<span id="cb25-30"><a href="#cb25-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-31"><a href="#cb25-31" aria-hidden="true" tabindex="-1"></a>**Lesson objectives:**   </span>
<span id="cb25-32"><a href="#cb25-32" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Learn to perform a search in academic literature database   </span>
<span id="cb25-33"><a href="#cb25-33" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Download search results and import them into R    </span>
<span id="cb25-34"><a href="#cb25-34" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Summarise bibliometric data   </span>
<span id="cb25-35"><a href="#cb25-35" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Make a few types of simple bibliometric networks   </span>
<span id="cb25-36"><a href="#cb25-36" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Plot bibliometric networks    </span>
<span id="cb25-37"><a href="#cb25-37" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb25-38"><a href="#cb25-38" aria-hidden="true" tabindex="-1"></a>**Lesson outline:**   </span>
<span id="cb25-39"><a href="#cb25-39" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>About this lesson   </span>
<span id="cb25-40"><a href="#cb25-40" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Getting bibliometric data   </span>
<span id="cb25-41"><a href="#cb25-41" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Summarising bibliometric data   </span>
<span id="cb25-42"><a href="#cb25-42" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>Creating bibliometric networks   </span>
<span id="cb25-43"><a href="#cb25-43" aria-hidden="true" tabindex="-1"></a><span class="ss"> * </span>More resources     </span>
<span id="cb25-44"><a href="#cb25-44" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb25-45"><a href="#cb25-45" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb25-46"><a href="#cb25-46" aria-hidden="true" tabindex="-1"></a>**** </span>
<span id="cb25-47"><a href="#cb25-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-48"><a href="#cb25-48" aria-hidden="true" tabindex="-1"></a><span class="fu">## About this lesson   </span></span>
<span id="cb25-49"><a href="#cb25-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-50"><a href="#cb25-50" aria-hidden="true" tabindex="-1"></a>This lesson is prepared for these who are already familiar with R coding language, R markdown and RStudio.</span>
<span id="cb25-51"><a href="#cb25-51" aria-hidden="true" tabindex="-1"></a>By the end of this tutorial you should be able to create a simple html document containing  markdown-formatted text, images and R code - all in RStudio.   </span>
<span id="cb25-52"><a href="#cb25-52" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-53"><a href="#cb25-53" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-54"><a href="#cb25-54" aria-hidden="true" tabindex="-1"></a>****</span>
<span id="cb25-55"><a href="#cb25-55" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-56"><a href="#cb25-56" aria-hidden="true" tabindex="-1"></a><span class="fu">## Getting bibliometric data</span></span>
<span id="cb25-57"><a href="#cb25-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-58"><a href="#cb25-58" aria-hidden="true" tabindex="-1"></a>You can do analyses of literature on any topic.   </span>
<span id="cb25-59"><a href="#cb25-59" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-60"><a href="#cb25-60" aria-hidden="true" tabindex="-1"></a>In this lesson we will have a look at the academic literature related to the concept of Terminal Investment.  </span>
<span id="cb25-61"><a href="#cb25-61" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-62"><a href="#cb25-62" aria-hidden="true" tabindex="-1"></a>Terminal Investment hypothesis predicts increased investment of resources into reproduction as the chances of survival decrease. This can be observed as increased reproductive effort in older animals or in animals challenged with factors signalling threat to their survival (e.g., predation, pathogenes, parasites).    </span>
<span id="cb25-63"><a href="#cb25-63" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-64"><a href="#cb25-64" aria-hidden="true" tabindex="-1"></a>Terminal investment in animals is usually studied in three main ways:     </span>
<span id="cb25-65"><a href="#cb25-65" aria-hidden="true" tabindex="-1"></a><span class="ss"> 1. </span>via observational studies of correlations of age and reproductive effort,      </span>
<span id="cb25-66"><a href="#cb25-66" aria-hidden="true" tabindex="-1"></a><span class="ss"> 2. </span>in experimental studies where animals are subject to immune challenges and their subsequent reproductive effort is compared to unchallenged animals of the same age,    </span>
<span id="cb25-67"><a href="#cb25-67" aria-hidden="true" tabindex="-1"></a><span class="ss"> 3. </span>in experimental studies where reproductive response to immune challenge is compared between animals of older ages versus younger ages.   </span>
<span id="cb25-68"><a href="#cb25-68" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-69"><a href="#cb25-69" aria-hidden="true" tabindex="-1"></a>You can read more on this Wiki page: <span class="ot">&lt;https://en.wikipedia.org/wiki/Terminal_investment_hypothesis&gt;</span>. We now actually completed a  meta-analysis on this topic and published our findings - you can find our published paper at <span class="ot">&lt;https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10176797/&gt;</span>).   </span>
<span id="cb25-70"><a href="#cb25-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-71"><a href="#cb25-71" aria-hidden="true" tabindex="-1"></a>We hope the topic is quite appealing and quiet easy to understand.There are several published reviews on terminal investment hypothesis and we can expect many publications related to this topic, as well as many researchers working on it. Is this so?  </span>
<span id="cb25-72"><a href="#cb25-72" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-73"><a href="#cb25-73" aria-hidden="true" tabindex="-1"></a>Today, we will try to run bibliometric analyses on the relevant sample of literature. Note that there are a few available  R packages (and many other online/software tools, and new are being developed) that can perform some of the tasks which we will practice during this exercise, and often they can do much more.    </span>
<span id="cb25-74"><a href="#cb25-74" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-75"><a href="#cb25-75" aria-hidden="true" tabindex="-1"></a>For your own project you may want to try to use some other tools, but there is no single "perfect" tool that fits all possible analyses and that is easy and usable for all disciplines and types of research questions. Also note that the main purpose of this exercise is to familiarize you with the basic principles/issues of bibliometric analyses. You can always learn more in your own time if you are interested.       </span>
<span id="cb25-76"><a href="#cb25-76" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-77"><a href="#cb25-77" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-78"><a href="#cb25-78" aria-hidden="true" tabindex="-1"></a>****</span>
<span id="cb25-79"><a href="#cb25-79" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-80"><a href="#cb25-80" aria-hidden="true" tabindex="-1"></a>*The search*</span>
<span id="cb25-81"><a href="#cb25-81" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-82"><a href="#cb25-82" aria-hidden="true" tabindex="-1"></a>First, we need to find a representative sample of academic publications on our topic of choice. For this, we will use cross-disciplinary database of academic literature, Scopus (<span class="ot">&lt;https://www.scopus.com&gt;</span>; owned by Elsevier). This database has the largest coverage of the published literature and should give us the most complete picture. An alternative database, commonly used for broad academic literature searches and analyses, is Web of Science: <span class="ot">&lt;https://www.webofknowledge.com/&gt;</span> (WoS; it has slightly different way of formatting search strings).   </span>
<span id="cb25-83"><a href="#cb25-83" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-84"><a href="#cb25-84" aria-hidden="true" tabindex="-1"></a>You should have free access to Scopus and WoS database on campus, but you may not be able to access it from outside the campus unless you use UNSW VPN or other university proxy servers (try logging in via the UNSW Library page, <span class="ot">&lt;https://www.library.unsw.edu.au&gt;</span>, then search for a link to Scopus database).   </span>
<span id="cb25-85"><a href="#cb25-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-86"><a href="#cb25-86" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-87"><a href="#cb25-87" aria-hidden="true" tabindex="-1"></a>Free-access broad-coverage databases include BASE (Bielfield Academic Search Engine, <span class="ot">&lt;https://www.base-search.net/&gt;</span>)  and Lens (https://www.lens.org). Google Scholar (<span class="ot">&lt;https://scholar.google.com/&gt;</span>) is not recommended for this type of work, mainly because it is hard to export bibliometric records from the results list.    </span>
<span id="cb25-88"><a href="#cb25-88" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-89"><a href="#cb25-89" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-90"><a href="#cb25-90" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-91"><a href="#cb25-91" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 1    </span></span>
<span id="cb25-92"><a href="#cb25-92" aria-hidden="true" tabindex="-1"></a>Go to Scopus search page: <span class="ot">&lt;https://www.scopus.com/search&gt;</span>. Careful, there are several different search engines in Scopus:</span>
<span id="cb25-93"><a href="#cb25-93" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Authors (also a new Researcher discovery) - information about authors (and affiliations).  </span>
<span id="cb25-94"><a href="#cb25-94" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Sources - information about journals (and other collections of documents).  </span>
<span id="cb25-95"><a href="#cb25-95" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Documents - information about journal articles (and other documents, such as book chapters) - WE WILL USE THIS ONE!  It has two interfaces: Basic and Advanced.    </span>
<span id="cb25-96"><a href="#cb25-96" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-97"><a href="#cb25-97" aria-hidden="true" tabindex="-1"></a>In the Document basic search (a simple single search window), enter the phrase "terminal investment" (without quotation marks) in the basic search window, as follows:   </span>
<span id="cb25-98"><a href="#cb25-98" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-101"><a href="#cb25-101" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-102"><a href="#cb25-102" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb25-103"><a href="#cb25-103" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-Scopus</span></span>
<span id="cb25-104"><a href="#cb25-104" aria-hidden="true" tabindex="-1"></a><span class="co">#| layout-ncol: 1</span></span>
<span id="cb25-105"><a href="#cb25-105" aria-hidden="true" tabindex="-1"></a><span class="co">#| column: page</span></span>
<span id="cb25-106"><a href="#cb25-106" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">"images/Scopus_TI_res_simple_search.png"</span>)</span>
<span id="cb25-107"><a href="#cb25-107" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-108"><a href="#cb25-108" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-109"><a href="#cb25-109" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-110"><a href="#cb25-110" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-111"><a href="#cb25-111" aria-hidden="true" tabindex="-1"></a>Hey, this does not look good... - many documents were found, and most of them are completely unrelated (e.g. on building shipping terminals).   </span>
<span id="cb25-112"><a href="#cb25-112" aria-hidden="true" tabindex="-1"></a>Why is that?    </span>
<span id="cb25-113"><a href="#cb25-113" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-114"><a href="#cb25-114" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-115"><a href="#cb25-115" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-116"><a href="#cb25-116" aria-hidden="true" tabindex="-1"></a>This is because our search is too simple. It allows us only to find the papers that explicitly mention terminal AND investment words in their title, abstract or keywords (the AND operator is a default one and it gets inserted automatically by the search software). To find a more relevant set of bibliographic records for our analyses, we need a more sophisticated search string.   </span>
<span id="cb25-117"><a href="#cb25-117" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-118"><a href="#cb25-118" aria-hidden="true" tabindex="-1"></a>Additionally, we will narrow down our topic a little bit more and aim to find papers that use immune challenge approach in wild or semi-wild animal species (so, we try to exclude established lab model species such as mice and rats, domesticated animals such as dogs and pigs, and humans). Finding the best search string is a bit of an art, so we just provide you with this one to save time (make sure to copy the whole thing - you may need to scroll to the right to see the complete string):   </span>
<span id="cb25-119"><a href="#cb25-119" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-120"><a href="#cb25-120" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-121"><a href="#cb25-121" aria-hidden="true" tabindex="-1"></a><span class="in">TITLE-ABS-KEY ( ( "terminal investment"  OR  "reproductive effort"  OR  "fecundity compensation"  OR  "reproductive compensation"  OR  "reproductive fitness"  OR  "reproductive investment"  OR  "reproductive success"  OR  "Life History Trade-Off*"  OR  "Phenotypic Plasticity" )  AND  ( "immune challeng*"  OR  "immunochalleng*"  OR  "infect*"  OR  lipopolysaccharide  OR  lps  OR  phytohemagglutinin  OR  pha  OR  "sheep red blood cells"  OR  srbc  OR  implant  OR  vaccin* ) )  AND NOT  TITLE-ABS-KEY ( load  OR  human  OR  people  OR  men  OR  women  OR  infant*  OR  rat  OR  rats  OR  mouse  OR  mice  OR  pig*  OR  pork  OR  beef  OR  cattle  OR  sheep  OR  lamb*  OR  chicken*  OR  calf*  OR  *horse* )  </span></span>
<span id="cb25-122"><a href="#cb25-122" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-123"><a href="#cb25-123" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-124"><a href="#cb25-124" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-125"><a href="#cb25-125" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-126"><a href="#cb25-126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-127"><a href="#cb25-127" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 2    </span></span>
<span id="cb25-128"><a href="#cb25-128" aria-hidden="true" tabindex="-1"></a>You need to copy and paste the above search string into the Advanced Search tab of the Scopus search page:</span>
<span id="cb25-129"><a href="#cb25-129" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-130"><a href="#cb25-130" aria-hidden="true" tabindex="-1"></a><span class="in">```{r Scopus_TI_res_all_adv_search.png, echo=FALSE, fig.cap=" ", out.width='80%'}</span></span>
<span id="cb25-131"><a href="#cb25-131" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb25-132"><a href="#cb25-132" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-Scopus_TI_search</span></span>
<span id="cb25-133"><a href="#cb25-133" aria-hidden="true" tabindex="-1"></a><span class="co">#| layout-ncol: 1</span></span>
<span id="cb25-134"><a href="#cb25-134" aria-hidden="true" tabindex="-1"></a><span class="co">#| column: page</span></span>
<span id="cb25-135"><a href="#cb25-135" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">"images/Scopus_TI_res_all_adv_search.png"</span>)</span>
<span id="cb25-136"><a href="#cb25-136" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-137"><a href="#cb25-137" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-138"><a href="#cb25-138" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-139"><a href="#cb25-139" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-140"><a href="#cb25-140" aria-hidden="true" tabindex="-1"></a>Press "Search"" button. You should see results roughly like this:   </span>
<span id="cb25-141"><a href="#cb25-141" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-142"><a href="#cb25-142" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-143"><a href="#cb25-143" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-144"><a href="#cb25-144" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-147"><a href="#cb25-147" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-148"><a href="#cb25-148" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb25-149"><a href="#cb25-149" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-Scopus_TI</span></span>
<span id="cb25-150"><a href="#cb25-150" aria-hidden="true" tabindex="-1"></a><span class="co">#| layout-ncol: 1</span></span>
<span id="cb25-151"><a href="#cb25-151" aria-hidden="true" tabindex="-1"></a><span class="co">#| column: page</span></span>
<span id="cb25-152"><a href="#cb25-152" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">"images/Scopus_TI_res_all.png"</span>)</span>
<span id="cb25-153"><a href="#cb25-153" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-154"><a href="#cb25-154" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-155"><a href="#cb25-155" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-156"><a href="#cb25-156" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-157"><a href="#cb25-157" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-158"><a href="#cb25-158" aria-hidden="true" tabindex="-1"></a>There should be over 1,400 records retrieved from the Scopus database. Some look relevant and many are not, but that is always the case, for a systematic review or a meta-analysis we would manually screen all and exclude irrelevant papers).    </span>
<span id="cb25-159"><a href="#cb25-159" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-160"><a href="#cb25-160" aria-hidden="true" tabindex="-1"></a>On the left side of the results window you can see simple filters: year, most common author names, subject areas, etc. You can explore the whole set roughly by using "Analyze search results" link above the table of the hits:   </span>
<span id="cb25-161"><a href="#cb25-161" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-164"><a href="#cb25-164" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-165"><a href="#cb25-165" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb25-166"><a href="#cb25-166" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-Scopus_TI_analyses</span></span>
<span id="cb25-167"><a href="#cb25-167" aria-hidden="true" tabindex="-1"></a><span class="co">#| layout-ncol: 1</span></span>
<span id="cb25-168"><a href="#cb25-168" aria-hidden="true" tabindex="-1"></a><span class="co">#| column: page</span></span>
<span id="cb25-169"><a href="#cb25-169" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">"images/Scopus_TI_res_analaysis.png"</span>)</span>
<span id="cb25-170"><a href="#cb25-170" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-171"><a href="#cb25-171" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-172"><a href="#cb25-172" aria-hidden="true" tabindex="-1"></a>There are more graphs below, and you can expand and customise them a bit too - have a look!   </span>
<span id="cb25-173"><a href="#cb25-173" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-174"><a href="#cb25-174" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-175"><a href="#cb25-175" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-176"><a href="#cb25-176" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-177"><a href="#cb25-177" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 3    </span></span>
<span id="cb25-178"><a href="#cb25-178" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-179"><a href="#cb25-179" aria-hidden="true" tabindex="-1"></a>Next, we can export the bibliographic records for more detailed bibliometric analyses in R. To do so, close the Scopus analyses window and go back to the list of the records found. First, select all records by clicking box "All" in the left top of the list of references. Then click the "Export" link to the right.    </span>
<span id="cb25-180"><a href="#cb25-180" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-181"><a href="#cb25-181" aria-hidden="true" tabindex="-1"></a>A pop-up window with the export options will appear.   </span>
<span id="cb25-182"><a href="#cb25-182" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-183"><a href="#cb25-183" aria-hidden="true" tabindex="-1"></a>First, select the format of the export: we will uses .bib file (BibTex format of references, one of the standard ones).   </span>
<span id="cb25-184"><a href="#cb25-184" aria-hidden="true" tabindex="-1"></a>Second, select which fields have to be exported by clicking the boxes on top of each column (or as needed).   </span>
<span id="cb25-185"><a href="#cb25-185" aria-hidden="true" tabindex="-1"></a>For bibliometric analyses on the citations among papers, it is essential to tick the box next to "Include references" (i.e. data on the cited documents).   </span>
<span id="cb25-186"><a href="#cb25-186" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-187"><a href="#cb25-187" aria-hidden="true" tabindex="-1"></a>Note that, unfortunately, Scopus limits number of exported records to 2,000. For longer lists of records, you will need to split them in smaller chunks for the export and then merge into a single larger data set (not covered in this tutorial; WoS export limits are 1,000 records).    </span>
<span id="cb25-188"><a href="#cb25-188" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-191"><a href="#cb25-191" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-192"><a href="#cb25-192" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb25-193"><a href="#cb25-193" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-Scopus_export</span></span>
<span id="cb25-194"><a href="#cb25-194" aria-hidden="true" tabindex="-1"></a><span class="co">#| layout-ncol: 1</span></span>
<span id="cb25-195"><a href="#cb25-195" aria-hidden="true" tabindex="-1"></a><span class="co">#| column: page</span></span>
<span id="cb25-196"><a href="#cb25-196" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">"images/Scopus_TI_res_all_export_bib.png"</span>)</span>
<span id="cb25-197"><a href="#cb25-197" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-198"><a href="#cb25-198" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-199"><a href="#cb25-199" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-200"><a href="#cb25-200" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-201"><a href="#cb25-201" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-202"><a href="#cb25-202" aria-hidden="true" tabindex="-1"></a>Click "Export" button. A file named "Scopus" (with extension matching your export type file, e.g., bib) will be saved to your downloads folder.    </span>
<span id="cb25-203"><a href="#cb25-203" aria-hidden="true" tabindex="-1"></a>Note that when you export references with their reference lists included in the records, the resulting files are quite large (in our case around 16Mb).   </span>
<span id="cb25-204"><a href="#cb25-204" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-205"><a href="#cb25-205" aria-hidden="true" tabindex="-1"></a>For this exercise, you can reduce your data set by applying some additional filters - e.g. including only literature from the last 5 years.      </span>
<span id="cb25-206"><a href="#cb25-206" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-207"><a href="#cb25-207" aria-hidden="true" tabindex="-1"></a>In case you did not succeed exporting the files (or wish to work with exactly the same ones we used, or you cannot access Scopus), the files downloaded on 27/05/2019 are provided (note that a standard way is to store them inside a "/data" sub-directory).   </span>
<span id="cb25-208"><a href="#cb25-208" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-209"><a href="#cb25-209" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-210"><a href="#cb25-210" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-211"><a href="#cb25-211" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-212"><a href="#cb25-212" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 4    </span></span>
<span id="cb25-213"><a href="#cb25-213" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-214"><a href="#cb25-214" aria-hidden="true" tabindex="-1"></a>Create a new Rmarkdown file to save your code (you can do this within new RStudio project).</span>
<span id="cb25-215"><a href="#cb25-215" aria-hidden="true" tabindex="-1"></a>Install and upload *bibliometrix* R package:</span>
<span id="cb25-216"><a href="#cb25-216" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-217"><a href="#cb25-217" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-220"><a href="#cb25-220" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-221"><a href="#cb25-221" aria-hidden="true" tabindex="-1"></a><span class="co">#| eval: false</span></span>
<span id="cb25-222"><a href="#cb25-222" aria-hidden="true" tabindex="-1"></a><span class="fu">install.packages</span>(<span class="st">"bibliometrix"</span>, <span class="at">dependencies=</span><span class="cn">TRUE</span>) <span class="do">### installs bibliometrix package and dependencies</span></span>
<span id="cb25-223"><a href="#cb25-223" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(bibliometrix)   <span class="co">#uploads the package</span></span>
<span id="cb25-224"><a href="#cb25-224" aria-hidden="true" tabindex="-1"></a><span class="co"># Note: output not displayed for this chunk</span></span>
<span id="cb25-225"><a href="#cb25-225" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-226"><a href="#cb25-226" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-227"><a href="#cb25-227" aria-hidden="true" tabindex="-1"></a>Upload the file exported from Scopus (you can use the one provided) into RStudio (note that the file path you may need to use on your computer may be different, e.g., "H:/Users/z1234567/Downloads/scopus.bib").    </span>
<span id="cb25-228"><a href="#cb25-228" aria-hidden="true" tabindex="-1"></a>Then, convert the data from that file into internal *bibliometrix* format.  </span>
<span id="cb25-229"><a href="#cb25-229" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-232"><a href="#cb25-232" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-233"><a href="#cb25-233" aria-hidden="true" tabindex="-1"></a>bib <span class="ot">&lt;-</span> <span class="fu">convert2df</span>(<span class="st">"data/scopus.bib"</span>, <span class="at">dbsource =</span> <span class="st">"scopus"</span>, <span class="at">format =</span> <span class="st">"bibtex"</span>) <span class="co"># Convert to a bibliometric data frame</span></span>
<span id="cb25-234"><a href="#cb25-234" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(bib)</span>
<span id="cb25-235"><a href="#cb25-235" aria-hidden="true" tabindex="-1"></a><span class="co">#write.csv(bib, "data/bib_as_df.csv", row.names = FALSE) #if you want to save this data frame as a csv file</span></span>
<span id="cb25-236"><a href="#cb25-236" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-237"><a href="#cb25-237" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-238"><a href="#cb25-238" aria-hidden="true" tabindex="-1"></a>After some processing, an object called "bib" is created. It contains a data frame with each row corresponding to one exported publication from Scopus and with each column corresponding to a field exported from Scopus online database. If you tried to achieve this by exporting a csv file directly from Scopus, you would sometimes get a messy data frame, due to missing field values shifting the cells between columns. </span>
<span id="cb25-239"><a href="#cb25-239" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-240"><a href="#cb25-240" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-241"><a href="#cb25-241" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-242"><a href="#cb25-242" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-243"><a href="#cb25-243" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-244"><a href="#cb25-244" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 5    </span></span>
<span id="cb25-245"><a href="#cb25-245" aria-hidden="true" tabindex="-1"></a>What are the contents of the columns of our "bib" data frame?  Columns are labelled with a two-letter tags: <span class="in">`r names(bib)`</span>.    </span>
<span id="cb25-246"><a href="#cb25-246" aria-hidden="true" tabindex="-1"></a>For a complete list and descriptions of field tags used in *bibliometrix* you can have a look at this file:   <span class="ot">&lt;http://www.bibliometrix.org/documents/Field_Tags_bibliometrix.pdf&gt;</span>.    </span>
<span id="cb25-247"><a href="#cb25-247" aria-hidden="true" tabindex="-1"></a>Our data frame contains just a subset of these codes. Which ones?   </span>
<span id="cb25-248"><a href="#cb25-248" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb25-249"><a href="#cb25-249" aria-hidden="true" tabindex="-1"></a>Note that column **bib$AU** contains authors of each paper (as surnames and initials) separated by semicolon (;). We can easily split these strings and extract a list of all author names to a vector:    </span>
<span id="cb25-250"><a href="#cb25-250" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-253"><a href="#cb25-253" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-254"><a href="#cb25-254" aria-hidden="true" tabindex="-1"></a><span class="co"># head(bib$AU) #have a look at the few few records on your screen</span></span>
<span id="cb25-255"><a href="#cb25-255" aria-hidden="true" tabindex="-1"></a>authors <span class="ot">&lt;-</span> bib<span class="sc">$</span>AU</span>
<span id="cb25-256"><a href="#cb25-256" aria-hidden="true" tabindex="-1"></a>authors <span class="ot">&lt;-</span> <span class="fu">unlist</span>(<span class="fu">strsplit</span>(authors, <span class="st">";"</span>)) <span class="co">#split the records into individual authors</span></span>
<span id="cb25-257"><a href="#cb25-257" aria-hidden="true" tabindex="-1"></a>authors <span class="ot">&lt;-</span> authors[<span class="fu">order</span>(authors)] <span class="co">#order alphabetically</span></span>
<span id="cb25-258"><a href="#cb25-258" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(authors) <span class="co">#have a look again</span></span>
<span id="cb25-259"><a href="#cb25-259" aria-hidden="true" tabindex="-1"></a><span class="co"># View(unique(authors)) #use to see all the values</span></span>
<span id="cb25-260"><a href="#cb25-260" aria-hidden="true" tabindex="-1"></a><span class="co"># write.csv(authors, "data/author_list_uncleaned.csv", row.names = FALSE) #if you want to save this data frame as a csv file</span></span>
<span id="cb25-261"><a href="#cb25-261" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-262"><a href="#cb25-262" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-263"><a href="#cb25-263" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-264"><a href="#cb25-264" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-265"><a href="#cb25-265" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-266"><a href="#cb25-266" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 6    </span></span>
<span id="cb25-267"><a href="#cb25-267" aria-hidden="true" tabindex="-1"></a>Cited references for each included paper are in the CR column of the "bib" data frame. They are in a single string, also separated by semicolon (;). We can have a look at them and check whether familiar names were cited, e.g.:   </span>
<span id="cb25-268"><a href="#cb25-268" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-271"><a href="#cb25-271" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-272"><a href="#cb25-272" aria-hidden="true" tabindex="-1"></a><span class="fu">dim</span>(bib) <span class="co">#dimensions of the data frame</span></span>
<span id="cb25-273"><a href="#cb25-273" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(bib) <span class="co">#names of the columns of the data frame</span></span>
<span id="cb25-274"><a href="#cb25-274" aria-hidden="true" tabindex="-1"></a><span class="co">#bib$CR[1] #display a list of cited references for the first paper in the data frame </span></span>
<span id="cb25-275"><a href="#cb25-275" aria-hidden="true" tabindex="-1"></a><span class="co">#(we are not displaying it in this document as it is a very long string! - examine it on your screen instead)</span></span>
<span id="cb25-276"><a href="#cb25-276" aria-hidden="true" tabindex="-1"></a><span class="co">#look whether some of these names are cited:</span></span>
<span id="cb25-277"><a href="#cb25-277" aria-hidden="true" tabindex="-1"></a><span class="fu">grep</span>(<span class="st">"NAKAGAWA, S."</span>, bib<span class="sc">$</span>CR) </span>
<span id="cb25-278"><a href="#cb25-278" aria-hidden="true" tabindex="-1"></a><span class="fu">grep</span>(<span class="st">"CORNWELL, W."</span>, bib<span class="sc">$</span>CR) </span>
<span id="cb25-279"><a href="#cb25-279" aria-hidden="true" tabindex="-1"></a>bib[<span class="fu">grep</span>(<span class="st">"CORNWELL, W."</span>, bib<span class="sc">$</span>CR), <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>)] <span class="co">#who is citing?</span></span>
<span id="cb25-280"><a href="#cb25-280" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-281"><a href="#cb25-281" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-282"><a href="#cb25-282" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-283"><a href="#cb25-283" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-284"><a href="#cb25-284" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-285"><a href="#cb25-285" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-286"><a href="#cb25-286" aria-hidden="true" tabindex="-1"></a><span class="fu">## Summarising bibliometric data</span></span>
<span id="cb25-287"><a href="#cb25-287" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-288"><a href="#cb25-288" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 7    </span></span>
<span id="cb25-289"><a href="#cb25-289" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-290"><a href="#cb25-290" aria-hidden="true" tabindex="-1"></a>Luckily, *bibliometrix* package  has a handy function that summarises the information contained in the "bib" data frame, so we can get some quick facts about our set of papers.   </span>
<span id="cb25-291"><a href="#cb25-291" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-292"><a href="#cb25-292" aria-hidden="true" tabindex="-1"></a>**Note:** this and the following tasks require quite a bit of computational power, they may be slow or even halt on your computer.    </span>
<span id="cb25-293"><a href="#cb25-293" aria-hidden="true" tabindex="-1"></a>In such case, for this exercise, make your data frame smaller by sub-setting it, e.g.:   </span>
<span id="cb25-294"><a href="#cb25-294" aria-hidden="true" tabindex="-1"></a>"bib &lt;- bib<span class="co">[</span><span class="ot">1:500, </span><span class="co">]</span> #taking first 500 records". However, the results and plots you will produce with a subsetted data frame will differ from the ones presented in this document.     </span>
<span id="cb25-295"><a href="#cb25-295" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-298"><a href="#cb25-298" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-299"><a href="#cb25-299" aria-hidden="true" tabindex="-1"></a><span class="co"># Preliminary descriptive analyses </span></span>
<span id="cb25-300"><a href="#cb25-300" aria-hidden="true" tabindex="-1"></a>results <span class="ot">&lt;-</span> <span class="fu">biblioAnalysis</span>(bib, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-301"><a href="#cb25-301" aria-hidden="true" tabindex="-1"></a><span class="co">#summary(object = results, k = 10, pause = TRUE) #display a series of summary tables</span></span>
<span id="cb25-302"><a href="#cb25-302" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-303"><a href="#cb25-303" aria-hidden="true" tabindex="-1"></a><span class="co"># Reduce the data set to the subset first 500 records</span></span>
<span id="cb25-304"><a href="#cb25-304" aria-hidden="true" tabindex="-1"></a>bib <span class="ot">&lt;-</span> bib[<span class="dv">1</span><span class="sc">:</span><span class="dv">500</span>, ] <span class="co">#taking first 500 records</span></span>
<span id="cb25-305"><a href="#cb25-305" aria-hidden="true" tabindex="-1"></a><span class="co">#summary(object = results, k = 10, pause = TRUE) #display a series of summary tables</span></span>
<span id="cb25-306"><a href="#cb25-306" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-307"><a href="#cb25-307" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-308"><a href="#cb25-308" aria-hidden="true" tabindex="-1"></a>Using *summary* function on *bibliometrix* results, you  get several screens with various tables summarising bibliometric data from our data frame - how many documents, journals, keywords, authors, publications time-span, collaboration index, annual publication growth rate, most prolific authors, publications per country, per journal, per keywords, etc.    </span>
<span id="cb25-309"><a href="#cb25-309" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-310"><a href="#cb25-310" aria-hidden="true" tabindex="-1"></a>You can automatically plot some of these tables (hit "return"" to display next graph, and later you can use arrows in the top left of the plots pane to move back and forth between consecutive plots saved in the RStudio memory):  </span>
<span id="cb25-311"><a href="#cb25-311" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-314"><a href="#cb25-314" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-315"><a href="#cb25-315" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false</span></span>
<span id="cb25-316"><a href="#cb25-316" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(results, <span class="at">k =</span> <span class="dv">10</span>, <span class="at">pause=</span><span class="cn">TRUE</span>) <span class="co">#this takes top 10 values from each plottable table</span></span>
<span id="cb25-317"><a href="#cb25-317" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-318"><a href="#cb25-318" aria-hidden="true" tabindex="-1"></a><span class="co">#the code below is for saving these plots into a pdf:</span></span>
<span id="cb25-319"><a href="#cb25-319" aria-hidden="true" tabindex="-1"></a><span class="co"># pdf(file = "plots/bib_descriptive_plots.pdf", height = 8, width = 8, pointsize=10) #</span></span>
<span id="cb25-320"><a href="#cb25-320" aria-hidden="true" tabindex="-1"></a><span class="co"># plot(results, k = 20, pause=FALSE) #this takes top 20 values from each plottable table</span></span>
<span id="cb25-321"><a href="#cb25-321" aria-hidden="true" tabindex="-1"></a><span class="co"># dev.off()</span></span>
<span id="cb25-322"><a href="#cb25-322" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-323"><a href="#cb25-323" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-324"><a href="#cb25-324" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-325"><a href="#cb25-325" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-326"><a href="#cb25-326" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-327"><a href="#cb25-327" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 8    </span></span>
<span id="cb25-328"><a href="#cb25-328" aria-hidden="true" tabindex="-1"></a>The cited papers from the CR field of the data frame can be analysed using function *citations*.   </span>
<span id="cb25-329"><a href="#cb25-329" aria-hidden="true" tabindex="-1"></a>Function *citations* makes it easy to generate the frequency tables of the most cited papers or the most cited first authors.    </span>
<span id="cb25-330"><a href="#cb25-330" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-331"><a href="#cb25-331" aria-hidden="true" tabindex="-1"></a>Ten most cited papers:   </span>
<span id="cb25-332"><a href="#cb25-332" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-335"><a href="#cb25-335" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-336"><a href="#cb25-336" aria-hidden="true" tabindex="-1"></a>mostcitedP <span class="ot">&lt;-</span> <span class="fu">citations</span>(bib, <span class="at">field =</span> <span class="st">"article"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-337"><a href="#cb25-337" aria-hidden="true" tabindex="-1"></a><span class="fu">cbind</span>(mostcitedP<span class="sc">$</span>Cited[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>]) <span class="co">#first 10</span></span>
<span id="cb25-338"><a href="#cb25-338" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-339"><a href="#cb25-339" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-340"><a href="#cb25-340" aria-hidden="true" tabindex="-1"></a>Ten most cited authors:</span>
<span id="cb25-341"><a href="#cb25-341" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-344"><a href="#cb25-344" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-345"><a href="#cb25-345" aria-hidden="true" tabindex="-1"></a>mostcitedA <span class="ot">&lt;-</span> <span class="fu">citations</span>(bib, <span class="at">field =</span> <span class="st">"author"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-346"><a href="#cb25-346" aria-hidden="true" tabindex="-1"></a><span class="fu">cbind</span>(mostcitedA<span class="sc">$</span>Cited[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>]) <span class="co">#first 10</span></span>
<span id="cb25-347"><a href="#cb25-347" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-348"><a href="#cb25-348" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-349"><a href="#cb25-349" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-350"><a href="#cb25-350" aria-hidden="true" tabindex="-1"></a>Function *localCitations* generates the frequency table of the **locally** most cited authors. Locally means that  citations are counted only within the given data set - i.e.  how many times an author/paper that is in this data set has been cited by other authors/papers also in the data set.</span>
<span id="cb25-351"><a href="#cb25-351" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-352"><a href="#cb25-352" aria-hidden="true" tabindex="-1"></a>Ten most frequent local cited authors and papers:</span>
<span id="cb25-353"><a href="#cb25-353" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-356"><a href="#cb25-356" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-357"><a href="#cb25-357" aria-hidden="true" tabindex="-1"></a>mostcitedLA <span class="ot">&lt;-</span> <span class="fu">localCitations</span>(bib, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-358"><a href="#cb25-358" aria-hidden="true" tabindex="-1"></a>mostcitedLA<span class="sc">$</span>Authors[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>,]</span>
<span id="cb25-359"><a href="#cb25-359" aria-hidden="true" tabindex="-1"></a>mostcitedLA<span class="sc">$</span>Papers[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>,]</span>
<span id="cb25-360"><a href="#cb25-360" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-361"><a href="#cb25-361" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-362"><a href="#cb25-362" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-363"><a href="#cb25-363" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-364"><a href="#cb25-364" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-365"><a href="#cb25-365" aria-hidden="true" tabindex="-1"></a><span class="fu">## Creating bibliometric networks</span></span>
<span id="cb25-366"><a href="#cb25-366" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-367"><a href="#cb25-367" aria-hidden="true" tabindex="-1"></a>So far, we looked only at the numbers - who or what gets cited most, either from the main papers list or from the lists of the references within these papers. Now it is time to look at the actual networks of citations and also other types of networks that can be created using our data set.   </span>
<span id="cb25-368"><a href="#cb25-368" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-369"><a href="#cb25-369" aria-hidden="true" tabindex="-1"></a>To do so, we will be creating various rectangular matrices which reflect connections of different attributes of Documents/Authors. These matrices than can be plotted as bipartite networks and analysed.  </span>
<span id="cb25-370"><a href="#cb25-370" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-371"><a href="#cb25-371" aria-hidden="true" tabindex="-1"></a>Co-citation or coupling networks are a special type of networks build from the information on scientific papers containing references to other scientific papers.   </span>
<span id="cb25-372"><a href="#cb25-372" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-373"><a href="#cb25-373" aria-hidden="true" tabindex="-1"></a>Package *bibliometrix* contains function *biblioNetwork* which makes creating bibliographic networks easy.  This function can create the most frequently used coupling networks: Authors, Sources, and Countries.   </span>
<span id="cb25-374"><a href="#cb25-374" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-375"><a href="#cb25-375" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-376"><a href="#cb25-376" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-377"><a href="#cb25-377" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-378"><a href="#cb25-378" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 9    </span></span>
<span id="cb25-379"><a href="#cb25-379" aria-hidden="true" tabindex="-1"></a>**Bibliographic coupling** - two articles are bibliographically coupled if they share at least one reference from their reference lists, i.e. at least one cited source appears in the reference lists/bibliographies of both papers (Kessler, 1963).    </span>
<span id="cb25-380"><a href="#cb25-380" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-383"><a href="#cb25-383" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-384"><a href="#cb25-384" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"coupling"</span>, <span class="at">network =</span> <span class="st">"references"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-385"><a href="#cb25-385" aria-hidden="true" tabindex="-1"></a>net <span class="ot">=</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted =</span> <span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Papers' bibliographic coupling"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span>
<span id="cb25-386"><a href="#cb25-386" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-387"><a href="#cb25-387" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-388"><a href="#cb25-388" aria-hidden="true" tabindex="-1"></a>Above, we plotted only the top 10 most coupled papers (n = 10). Try increasing this number to 100 (would not recommend further increasing the number of displayed nodes - it gets slow and messy).   </span>
<span id="cb25-389"><a href="#cb25-389" aria-hidden="true" tabindex="-1"></a>What happens and why?    </span>
<span id="cb25-390"><a href="#cb25-390" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-391"><a href="#cb25-391" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb25-392"><a href="#cb25-392" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-393"><a href="#cb25-393" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-394"><a href="#cb25-394" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 10    </span></span>
<span id="cb25-395"><a href="#cb25-395" aria-hidden="true" tabindex="-1"></a>**Author's bibliographic coupling** - two authors are bibliographically coupled if they share at least one reference form their reference lists.    </span>
<span id="cb25-396"><a href="#cb25-396" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-399"><a href="#cb25-399" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-400"><a href="#cb25-400" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"coupling"</span>, <span class="at">network =</span> <span class="st">"authors"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-401"><a href="#cb25-401" aria-hidden="true" tabindex="-1"></a>net <span class="ot">=</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted =</span> <span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Authors' bibliographic coupling"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.8</span>)</span>
<span id="cb25-402"><a href="#cb25-402" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-403"><a href="#cb25-403" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-404"><a href="#cb25-404" aria-hidden="true" tabindex="-1"></a>Above, we plotted only the top 10 most coupled authors (n = 10). Try increasing this number to 100 (would not recommend further increasing the number of displayed nodes to &gt;50 - it gets slow and messy).   </span>
<span id="cb25-405"><a href="#cb25-405" aria-hidden="true" tabindex="-1"></a>What happens and why?    </span>
<span id="cb25-406"><a href="#cb25-406" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-407"><a href="#cb25-407" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb25-408"><a href="#cb25-408" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-409"><a href="#cb25-409" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-410"><a href="#cb25-410" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 11     </span></span>
<span id="cb25-411"><a href="#cb25-411" aria-hidden="true" tabindex="-1"></a>**Bibliographic co-citation** is kind of opposite to bibliographic coupling, in so that two papers are linked by co-citation when both are cited in a third paper.    </span>
<span id="cb25-412"><a href="#cb25-412" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-415"><a href="#cb25-415" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-416"><a href="#cb25-416" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib[<span class="dv">1</span><span class="sc">:</span><span class="dv">50</span>,], <span class="at">analysis =</span> <span class="st">"co-citation"</span>, <span class="at">network =</span> <span class="st">"references"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-417"><a href="#cb25-417" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted=</span><span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Papers' co-citations"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span>
<span id="cb25-418"><a href="#cb25-418" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-419"><a href="#cb25-419" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-420"><a href="#cb25-420" aria-hidden="true" tabindex="-1"></a>Note that for creating this matrix we only used first 50 papers from our data set - this is because the resulting matrix is a matrix of ALL cited papers and it gets HUGE). Also, we plotted only the top 10 most coupled papers (n=10). Try increasing this number to 20 (would not recommend further increasing the number of displayed nodes to &gt;50 - it gets slow and messy).   </span>
<span id="cb25-421"><a href="#cb25-421" aria-hidden="true" tabindex="-1"></a>What happens and why?    </span>
<span id="cb25-422"><a href="#cb25-422" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-423"><a href="#cb25-423" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-424"><a href="#cb25-424" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-425"><a href="#cb25-425" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-426"><a href="#cb25-426" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 12     </span></span>
<span id="cb25-427"><a href="#cb25-427" aria-hidden="true" tabindex="-1"></a>**Bibliographic collaboration** is a network where nodes are authors and links are co-authorships of the papers.     </span>
<span id="cb25-428"><a href="#cb25-428" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-431"><a href="#cb25-431" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-432"><a href="#cb25-432" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"collaboration"</span>, <span class="at">network =</span> <span class="st">"authors"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-433"><a href="#cb25-433" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">weighted =</span> <span class="cn">NULL</span>, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Authors' collaborations"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> <span class="dv">5</span>, <span class="at">remove.multiple =</span> <span class="cn">TRUE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span>
<span id="cb25-434"><a href="#cb25-434" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-435"><a href="#cb25-435" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-436"><a href="#cb25-436" aria-hidden="true" tabindex="-1"></a>Above, we plotted only the top 10 most collaborating authors (n = 10). Try increasing this number to 100 (would not recommend further increasing the number of displayed nodes - it gets slow and messy).   </span>
<span id="cb25-437"><a href="#cb25-437" aria-hidden="true" tabindex="-1"></a>What happens and why?    </span>
<span id="cb25-438"><a href="#cb25-438" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-439"><a href="#cb25-439" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-440"><a href="#cb25-440" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-441"><a href="#cb25-441" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-442"><a href="#cb25-442" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 13     </span></span>
<span id="cb25-443"><a href="#cb25-443" aria-hidden="true" tabindex="-1"></a>**Country Scientific Collaboration** - we can visualise authors from which countries publish papers together most frequently.   </span>
<span id="cb25-444"><a href="#cb25-444" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-447"><a href="#cb25-447" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-448"><a href="#cb25-448" aria-hidden="true" tabindex="-1"></a>bib <span class="ot">&lt;-</span> <span class="fu">metaTagExtraction</span>(bib, <span class="at">Field =</span> <span class="st">"AU_CO"</span>, <span class="at">sep =</span> <span class="st">";"</span>) <span class="co">#we need to extract countries from the affiliations first</span></span>
<span id="cb25-449"><a href="#cb25-449" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"collaboration"</span>, <span class="at">network =</span> <span class="st">"countries"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-450"><a href="#cb25-450" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">n =</span> <span class="dv">10</span>, <span class="at">Title =</span> <span class="st">"Country Collaboration"</span>, <span class="at">type =</span> <span class="st">"auto"</span>, <span class="at">size =</span> <span class="cn">TRUE</span>, <span class="at">remove.multiple =</span> <span class="cn">FALSE</span>, <span class="at">labelsize =</span> <span class="fl">0.5</span>)</span>
<span id="cb25-451"><a href="#cb25-451" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-452"><a href="#cb25-452" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-453"><a href="#cb25-453" aria-hidden="true" tabindex="-1"></a>Above, we plotted only the top 10 most collaborating countries (n = 10). Try increasing this number to 50 (would not recommend further increasing the number of displayed nodes to &gt;100 - it gets slow and messy).   </span>
<span id="cb25-454"><a href="#cb25-454" aria-hidden="true" tabindex="-1"></a>What happens and why? </span>
<span id="cb25-455"><a href="#cb25-455" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-456"><a href="#cb25-456" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-457"><a href="#cb25-457" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-458"><a href="#cb25-458" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-459"><a href="#cb25-459" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 14     </span></span>
<span id="cb25-460"><a href="#cb25-460" aria-hidden="true" tabindex="-1"></a>**Keyword co-occurrences** - we can also visualise which papers share most keywords (from the Scopus database).    </span>
<span id="cb25-461"><a href="#cb25-461" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-464"><a href="#cb25-464" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-465"><a href="#cb25-465" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(bib, <span class="at">analysis =</span> <span class="st">"co-occurrences"</span>, <span class="at">network =</span> <span class="st">"keywords"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-466"><a href="#cb25-466" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">n =</span> <span class="dv">50</span>, <span class="at">Title =</span> <span class="st">"Keyword co-occurance"</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size =</span> T, <span class="at">remove.multiple =</span> <span class="cn">FALSE</span>, <span class="at">labelsize =</span> <span class="fl">0.7</span>, <span class="at">edgesize =</span> <span class="dv">5</span>)</span>
<span id="cb25-467"><a href="#cb25-467" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-468"><a href="#cb25-468" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-469"><a href="#cb25-469" aria-hidden="true" tabindex="-1"></a>Try replacing network = "keywords" with network = "author_keywords" and see what happens. You can also try to display fewer/more keywords in the plot.   </span>
<span id="cb25-470"><a href="#cb25-470" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-471"><a href="#cb25-471" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-472"><a href="#cb25-472" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-473"><a href="#cb25-473" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-474"><a href="#cb25-474" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 15  </span></span>
<span id="cb25-475"><a href="#cb25-475" aria-hidden="true" tabindex="-1"></a>**Note**: you may want to skip this step on a big data set or a slow computer.     </span>
<span id="cb25-476"><a href="#cb25-476" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-477"><a href="#cb25-477" aria-hidden="true" tabindex="-1"></a>**Co-Word Analysis** - uses the word co-occurrences in a bibliographic collection to map the conceptual structure of research. It works via a separate function *conceptualStructure* that creates a conceptual structure map of a scientific field performing Correspondence Analysis (CA), Multiple Correspondence Analysis (MCA) or Metric Multidimensional Scaling (MDS) and Clustering of a bipartite network of terms extracted from keyword, title or abstract fields of the data frame. Note that three different graphs are created as an output.   </span>
<span id="cb25-478"><a href="#cb25-478" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-481"><a href="#cb25-481" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-482"><a href="#cb25-482" aria-hidden="true" tabindex="-1"></a>CS <span class="ot">&lt;-</span> <span class="fu">conceptualStructure</span>(bib, <span class="at">field =</span> <span class="st">"ID"</span>, <span class="at">minDegree =</span> <span class="dv">20</span>, <span class="at">k.max =</span> <span class="dv">5</span>, <span class="at">stemming =</span> <span class="cn">FALSE</span>, <span class="at">labelsize =</span> <span class="dv">10</span>)</span>
<span id="cb25-483"><a href="#cb25-483" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-484"><a href="#cb25-484" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-485"><a href="#cb25-485" aria-hidden="true" tabindex="-1"></a>The code above uses field = "ID". ID stands for "Keywords Plus associated by ISI or SCOPUS database", to analyse the "conceptualStructure". You could try using authors keywords, "DE" field, instead of "ID".   </span>
<span id="cb25-486"><a href="#cb25-486" aria-hidden="true" tabindex="-1"></a>Is the new map different?    </span>
<span id="cb25-487"><a href="#cb25-487" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-488"><a href="#cb25-488" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-489"><a href="#cb25-489" aria-hidden="true" tabindex="-1"></a>****  </span>
<span id="cb25-490"><a href="#cb25-490" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-491"><a href="#cb25-491" aria-hidden="true" tabindex="-1"></a><span class="fu">### TASK 16   </span></span>
<span id="cb25-492"><a href="#cb25-492" aria-hidden="true" tabindex="-1"></a>**Note**: you may want to skip this step on a big data set or a slow computer.       </span>
<span id="cb25-493"><a href="#cb25-493" aria-hidden="true" tabindex="-1"></a>**Historical Direct Citation Network** - represents a chronological network map of most relevant direct citations in a bibliographic collection, i.e who is citing whom and in what order.</span>
<span id="cb25-494"><a href="#cb25-494" aria-hidden="true" tabindex="-1"></a>*histNetwork* function calculates a chronological direct citation network matrix which then is plotted using *histPlot*:</span>
<span id="cb25-495"><a href="#cb25-495" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-498"><a href="#cb25-498" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-499"><a href="#cb25-499" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false</span></span>
<span id="cb25-500"><a href="#cb25-500" aria-hidden="true" tabindex="-1"></a><span class="co">#options(width = 130)</span></span>
<span id="cb25-501"><a href="#cb25-501" aria-hidden="true" tabindex="-1"></a>histResults <span class="ot">&lt;-</span> <span class="fu">histNetwork</span>(bib, <span class="at">min.citations =</span> <span class="dv">10</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-502"><a href="#cb25-502" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">histPlot</span>(histResults, <span class="at">labelsize =</span> <span class="dv">2</span>)</span>
<span id="cb25-503"><a href="#cb25-503" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-504"><a href="#cb25-504" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-505"><a href="#cb25-505" aria-hidden="true" tabindex="-1"></a>Only articles with minimum of 10 citations are included in the above analysis, if you change this number to a higher value, the analyses will be quicker and the plot less dense. Have a try!      </span>
<span id="cb25-506"><a href="#cb25-506" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-507"><a href="#cb25-507" aria-hidden="true" tabindex="-1"></a>****</span>
<span id="cb25-508"><a href="#cb25-508" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-509"><a href="#cb25-509" aria-hidden="true" tabindex="-1"></a>**MORE TO DO**    </span>
<span id="cb25-510"><a href="#cb25-510" aria-hidden="true" tabindex="-1"></a>You can use different types of network plots - just tweak "type" parameter in the *networkPlot* function (check the vignette for the available options). "Type" parameter indicates the network map plotting layout: circle, kamada-kawai, mds, etc.     </span>
<span id="cb25-511"><a href="#cb25-511" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-512"><a href="#cb25-512" aria-hidden="true" tabindex="-1"></a>You can use non-R tools to visualise bibliographic networks, e.g. VOSviewer software by Nees Jan van Eck and Ludo Waltman (<span class="ot">&lt;http://www.vosviewer.com&gt;</span>). When in R function you use type=“vosviewer”, the function will export the network as a standard "pajek" network file (named “vosnetwork.net”), which can then be used in other network-plotting software, including VOSviewer.   </span>
<span id="cb25-513"><a href="#cb25-513" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-514"><a href="#cb25-514" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-515"><a href="#cb25-515" aria-hidden="true" tabindex="-1"></a>****   </span>
<span id="cb25-516"><a href="#cb25-516" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-517"><a href="#cb25-517" aria-hidden="true" tabindex="-1"></a><span class="fu">## Resources   </span></span>
<span id="cb25-518"><a href="#cb25-518" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-519"><a href="#cb25-519" aria-hidden="true" tabindex="-1"></a><span class="ss">* </span>Bibliometrix webpage (includes an online "biblioshiny" app): <span class="ot">&lt;http://www.bibliometrix.org/&gt;</span>.        </span>
<span id="cb25-520"><a href="#cb25-520" aria-hidden="true" tabindex="-1"></a><span class="ss">* </span>A more detailed *bibliometrix* information and links on GitHub: <span class="ot">&lt;https://github.com/massimoaria/bibliometrix&gt;</span>.    </span>
<span id="cb25-521"><a href="#cb25-521" aria-hidden="true" tabindex="-1"></a><span class="ss">* </span>Another useful R package - Revtools:  <span class="ot">&lt;http://revtools.net/&gt;</span>.   </span>
<span id="cb25-522"><a href="#cb25-522" aria-hidden="true" tabindex="-1"></a><span class="ss">* </span>Stand-alone GUI software (introduced above) - VOSviewer: <span class="ot">&lt;http://www.vosviewer.com/&gt;</span>.    </span>
<span id="cb25-523"><a href="#cb25-523" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-524"><a href="#cb25-524" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-527"><a href="#cb25-527" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-528"><a href="#cb25-528" aria-hidden="true" tabindex="-1"></a><span class="co">#| eval: false</span></span>
<span id="cb25-529"><a href="#cb25-529" aria-hidden="true" tabindex="-1"></a><span class="co">#| include: false</span></span>
<span id="cb25-530"><a href="#cb25-530" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-531"><a href="#cb25-531" aria-hidden="true" tabindex="-1"></a><span class="co">#source network</span></span>
<span id="cb25-532"><a href="#cb25-532" aria-hidden="true" tabindex="-1"></a>A <span class="ot">&lt;-</span> <span class="fu">cocMatrix</span>(bib, <span class="at">Field =</span> <span class="st">"SO"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-533"><a href="#cb25-533" aria-hidden="true" tabindex="-1"></a><span class="fu">sort</span>(Matrix<span class="sc">::</span><span class="fu">colSums</span>(A), <span class="at">decreasing =</span> <span class="cn">TRUE</span>)[<span class="dv">1</span><span class="sc">:</span><span class="dv">5</span>]</span>
<span id="cb25-534"><a href="#cb25-534" aria-hidden="true" tabindex="-1"></a><span class="fu">str</span>(A)</span>
<span id="cb25-535"><a href="#cb25-535" aria-hidden="true" tabindex="-1"></a><span class="co">#Citation network</span></span>
<span id="cb25-536"><a href="#cb25-536" aria-hidden="true" tabindex="-1"></a>A <span class="ot">&lt;-</span> <span class="fu">cocMatrix</span>(bib, <span class="at">Field =</span> <span class="st">"CR"</span>, <span class="at">sep =</span> <span class="st">".  "</span>)</span>
<span id="cb25-537"><a href="#cb25-537" aria-hidden="true" tabindex="-1"></a><span class="co">#Author network</span></span>
<span id="cb25-538"><a href="#cb25-538" aria-hidden="true" tabindex="-1"></a>A <span class="ot">&lt;-</span> <span class="fu">cocMatrix</span>(bib, <span class="at">Field =</span> <span class="st">"AU"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-539"><a href="#cb25-539" aria-hidden="true" tabindex="-1"></a><span class="co">#Country network</span></span>
<span id="cb25-540"><a href="#cb25-540" aria-hidden="true" tabindex="-1"></a><span class="co">#Authors’ Countries is not a standard attribute of the bibliographic data frame. You need to extract this information from affiliation attribute using the function metaTagExtraction.</span></span>
<span id="cb25-541"><a href="#cb25-541" aria-hidden="true" tabindex="-1"></a>M <span class="ot">&lt;-</span> <span class="fu">metaTagExtraction</span>(bib, <span class="at">Field =</span> <span class="st">"AU_CO"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-542"><a href="#cb25-542" aria-hidden="true" tabindex="-1"></a>A <span class="ot">&lt;-</span> <span class="fu">cocMatrix</span>(bib, <span class="at">Field =</span> <span class="st">"AU_CO"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-543"><a href="#cb25-543" aria-hidden="true" tabindex="-1"></a><span class="co">#metaTagExtraction allows to extract the following additional field tags: Authors’ countries (Field = "AU_CO"); First Author’s countries (Field = "AU_CO"); First author of each cited reference (Field = "CR_AU"); Publication source of each cited reference (Field = "CR_SO"); and Authors’ affiliations (Field = "AU_UN").</span></span>
<span id="cb25-544"><a href="#cb25-544" aria-hidden="true" tabindex="-1"></a><span class="co">#Author keyword network</span></span>
<span id="cb25-545"><a href="#cb25-545" aria-hidden="true" tabindex="-1"></a>A <span class="ot">&lt;-</span> <span class="fu">cocMatrix</span>(bib, <span class="at">Field =</span> <span class="st">"DE"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-546"><a href="#cb25-546" aria-hidden="true" tabindex="-1"></a><span class="co">#Keyword Plus network</span></span>
<span id="cb25-547"><a href="#cb25-547" aria-hidden="true" tabindex="-1"></a>A <span class="ot">&lt;-</span> <span class="fu">cocMatrix</span>(bib, <span class="at">Field =</span> <span class="st">"ID"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-548"><a href="#cb25-548" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb25-549"><a href="#cb25-549" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-550"><a href="#cb25-550" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-553"><a href="#cb25-553" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb25-554"><a href="#cb25-554" aria-hidden="true" tabindex="-1"></a><span class="co">#| eval: false</span></span>
<span id="cb25-555"><a href="#cb25-555" aria-hidden="true" tabindex="-1"></a><span class="co">#| include: false</span></span>
<span id="cb25-556"><a href="#cb25-556" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-557"><a href="#cb25-557" aria-hidden="true" tabindex="-1"></a><span class="do">#####################  Figure - CO-AUTHORS NETWORK  ####################################################</span></span>
<span id="cb25-558"><a href="#cb25-558" aria-hidden="true" tabindex="-1"></a><span class="fu">pdf</span>(<span class="at">file=</span><span class="st">"Figure_collaboration_network.pdf"</span>, <span class="at">width=</span><span class="dv">8</span>, <span class="at">height=</span><span class="dv">8</span>, <span class="at">pointsize=</span><span class="dv">10</span>)</span>
<span id="cb25-559"><a href="#cb25-559" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mfrow=</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">1</span>), <span class="at">mar=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">0</span>,<span class="dv">0</span>,<span class="dv">0</span>))</span>
<span id="cb25-560"><a href="#cb25-560" aria-hidden="true" tabindex="-1"></a>M <span class="ot">&lt;-</span> <span class="fu">metaTagExtraction</span>(bib, <span class="at">Field =</span> <span class="st">"CR_AU"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-561"><a href="#cb25-561" aria-hidden="true" tabindex="-1"></a>NetMatrix <span class="ot">&lt;-</span> <span class="fu">biblioNetwork</span>(M, <span class="at">analysis =</span> <span class="st">"collaboration"</span>, <span class="at">network =</span> <span class="st">"authors"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-562"><a href="#cb25-562" aria-hidden="true" tabindex="-1"></a>net <span class="ot">&lt;-</span> <span class="fu">networkPlot</span>(NetMatrix, <span class="at">n =</span> <span class="dv">100</span>, <span class="at">cluster=</span><span class="st">"walktrap"</span>, <span class="at">Title =</span> <span class="st">""</span>, <span class="at">labelsize =</span> <span class="dv">1</span>, <span class="at">type =</span> <span class="st">"fruchterman"</span>, <span class="at">size=</span><span class="cn">TRUE</span>, <span class="at">remove.multiple=</span><span class="cn">TRUE</span>)    </span>
<span id="cb25-563"><a href="#cb25-563" aria-hidden="true" tabindex="-1"></a><span class="fu">dev.off</span>() <span class="co">#note, changing n alters the colours and arrangements of the authors and clusters</span></span>
<span id="cb25-564"><a href="#cb25-564" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-565"><a href="#cb25-565" aria-hidden="true" tabindex="-1"></a><span class="do">#####################  Figure - HISTORICAL CO-CITATION NETWORK  ####################################################</span></span>
<span id="cb25-566"><a href="#cb25-566" aria-hidden="true" tabindex="-1"></a><span class="fu">pdf</span>(<span class="at">file=</span><span class="st">"Figure_historical_network.pdf"</span>, <span class="at">width=</span><span class="dv">8</span>, <span class="at">height=</span><span class="dv">8</span>, <span class="at">pointsize=</span><span class="dv">10</span>)</span>
<span id="cb25-567"><a href="#cb25-567" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mfrow=</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">1</span>), <span class="at">mar=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">0</span>,<span class="dv">0</span>,<span class="dv">0</span>))</span>
<span id="cb25-568"><a href="#cb25-568" aria-hidden="true" tabindex="-1"></a>M <span class="ot">&lt;-</span> <span class="fu">metaTagExtraction</span>(bib, <span class="at">Field =</span> <span class="st">"CR_AU"</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb25-569"><a href="#cb25-569" aria-hidden="true" tabindex="-1"></a>histResults <span class="ot">&lt;-</span> <span class="fu">histNetwork</span>(M, <span class="at">n=</span><span class="dv">143</span>, <span class="at">sep =</span> <span class="st">";"</span>) <span class="co"># "n" (number of most cited references to select) picked manually, larger numbers give error</span></span>
<span id="cb25-570"><a href="#cb25-570" aria-hidden="true" tabindex="-1"></a><span class="fu">histPlot</span>(histResults, <span class="at">size=</span><span class="cn">TRUE</span>) </span>
<span id="cb25-571"><a href="#cb25-571" aria-hidden="true" tabindex="-1"></a><span class="fu">dev.off</span>()</span>
<span id="cb25-572"><a href="#cb25-572" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->



</body></html>